{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 1. Making Data-structure Neutral Queries by URI\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##### Use Case Scenario1 : Query for the OMIM ID for a Variant Using myvariant.info"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Approach 1: Using Myvariant.info"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Users could retrieve the OMIM ID through myvariant.info; however, it requires the user to understand how OMIM ID is embedded in the data structure of MyVariant.info."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "# import myvariant python package\n",
    "import myvariant\n",
    "mv = myvariant.MyVariantInfo()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'_id': 'chr9:g.135781006_135781007del',\n",
       " '_version': 1,\n",
       " 'clinvar': {'rcv': {'conditions': {'identifiers': {'omim': '109800'}}}}}"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Fetch info about OMIM ID\n",
    "mv.getvariant('chr9:g.135781006_135781007del', fields='clinvar.rcv.conditions.identifiers.omim')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Approach 2: Using JSON-LD powered neutral query function"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "By utilizing JSON-LD in making queries, the process would be simplified significantly. Users only need to know the URI for OMIM ID, which is unique for any API. And it saves users significant amount of time in order to figure out the data structure for each API."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "'''\n",
    "import get_biothings function which is built utilizing JSON-LD technology\n",
    "This function could be used to make neutral query for all BioThings APIs,\n",
    "e.g MyGene.info, MyVariant.info, Drug and compound API\n",
    "'''\n",
    "from biothings_helper import get_biothings"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'_id': 'chr9:g.135781006_135781007del',\n",
       " '_version': 1,\n",
       " 'clinvar': {'rcv': {'conditions': {'identifiers': {'omim': '109800'}}}}}"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Fetch info about OMIM ID using URI for OMIM ID, which is 'http://identifiers.org/omim/'\n",
    "get_biothings(api='myvariant.info',id='chr9:g.135781006_135781007del', fields_uri='http://identifiers.org/omim/')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##### Use Case Scenario2 : query for all variants in MyVariant.info related to gene CDK2"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Approach 1: Using MyVariant.info"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The traditional approach would first require the user to figure out which data source in MyVariant.info includes annotation information about gene names. Additionally, the user have to understand how gene name is represented and embedded in each of the data source."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'hits': [{'_id': 'chr12:g.56361693G>T',\n",
       "   '_score': 8.73773,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'G',\n",
       "    'annotype': ['CodingTranscript', 'Intergenic', 'Intergenic'],\n",
       "    'bstatistic': 251,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.024,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.039,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.142,\n",
       "     'tssaflnk': 0.291,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.0,\n",
       "     'txflnk': 0.063,\n",
       "     'txwk': 0.181,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': ['stop_gained', 'upstream', 'upstream'],\n",
       "    'consequence': ['STOP_GAINED', 'UPSTREAM', 'UPSTREAM'],\n",
       "    'consscore': [8, 1, 1],\n",
       "    'cpg': 0.03,\n",
       "    'dna': {'helt': 0.43, 'mgw': 0.6, 'prot': -1.24, 'roll': 6.36},\n",
       "    'encode': {'exp': 521.18,\n",
       "     'h3k27ac': 20.96,\n",
       "     'h3k4me1': 29.52,\n",
       "     'h3k4me3': 55.04,\n",
       "     'nucleo': 2.1},\n",
       "    'exon': '2/7',\n",
       "    'fitcons': 0.623825,\n",
       "    'gc': 0.48,\n",
       "    'gene': [{'ccds_id': 'CCDS8898.1',\n",
       "      'cds': {'cdna_pos': 409,\n",
       "       'cds_pos': 169,\n",
       "       'rel_cdna_pos': 0.18,\n",
       "       'rel_cds_pos': 0.19},\n",
       "      'feature_id': 'ENST00000266970',\n",
       "      'gene_id': 'ENSG00000123374',\n",
       "      'genename': 'CDK2',\n",
       "      'prot': {'domain': 'ndomain', 'protpos': 57, 'rel_prot_pos': 0.19}},\n",
       "     {'ccds_id': 'CCDS55834.1',\n",
       "      'feature_id': 'ENST00000449260',\n",
       "      'gene_id': 'ENSG00000185664',\n",
       "      'genename': 'PMEL'},\n",
       "     {'feature_id': 'ENST00000554022',\n",
       "      'gene_id': 'ENSG00000258554',\n",
       "      'genename': 'RP11-973D8.4'}],\n",
       "    'gerp': {'n': 4.73, 'rs': 406.8, 'rs_pval': 1.65278e-39, 's': 4.73},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 403,\n",
       "    'min_dist_tss': 435,\n",
       "    'mutindex': -42,\n",
       "    'naa': '*',\n",
       "    'oaa': 'E',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.996, 'vertebrate': 1.0},\n",
       "    'phred': 38,\n",
       "    'phylop': {'mammalian': 2.626, 'primate': 0.486, 'vertebrate': 5.559},\n",
       "    'pos': 56361693,\n",
       "    'rawscore': 11.874943,\n",
       "    'ref': 'G',\n",
       "    'segway': 'E/GM',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 1,\n",
       "     'pos': 57,\n",
       "     'ref': 'E',\n",
       "     'refcodon': 'GAG'},\n",
       "    'alt': 'T',\n",
       "    'ancestral_allele': 'G',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.7981,\n",
       "     'score': 0.9969183004977041},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 22.49633,\n",
       "     'raw': 1.18564590828305},\n",
       "    'eigen-pc': {'phred': 20.46959,\n",
       "     'raw': 1.02513172599092,\n",
       "     'raw_rankscore': 0.9855},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970', 'ENSP00000452514', 'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000354056']},\n",
       "    'exac': {'ac': 1,\n",
       "     'adj_ac': 1,\n",
       "     'adj_af': 8.238e-06,\n",
       "     'af': 8.236e-06,\n",
       "     'afr_ac': 0,\n",
       "     'afr_af': 0,\n",
       "     'amr_ac': 0,\n",
       "     'amr_af': 0,\n",
       "     'eas_ac': 0,\n",
       "     'eas_af': 0,\n",
       "     'fin_ac': 0,\n",
       "     'fin_af': 0,\n",
       "     'nfe_ac': 1,\n",
       "     'nfe_af': 1.499e-05,\n",
       "     'sas_ac': 0,\n",
       "     'sas_af': 0},\n",
       "    'exac_nontcga': {'ac': 1,\n",
       "     'adj_ac': 1,\n",
       "     'adj_af': 9.417e-06,\n",
       "     'af': 9.415e-06,\n",
       "     'afr_ac': 0,\n",
       "     'afr_af': 0,\n",
       "     'amr_ac': 0,\n",
       "     'amr_af': 0,\n",
       "     'eas_ac': 0,\n",
       "     'eas_af': 0,\n",
       "     'fin_ac': 0,\n",
       "     'fin_af': 0,\n",
       "     'nfe_ac': 1,\n",
       "     'nfe_af': 1.84e-05,\n",
       "     'sas_ac': 0,\n",
       "     'sas_af': 0},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFBCI',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.70836,\n",
       "     'coding_score': 0.96805},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999999999985},\n",
       "    'gerp++': {'nr': 4.73, 'rs': 4.73, 'rs_rankscore': 0.59331},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.61804,\n",
       "     'fitcons_score': 0.685571},\n",
       "    'hg18': {'end': 54647960, 'start': 54647960},\n",
       "    'hg19': {'end': 56361693, 'start': 56361693},\n",
       "    'hg38': {'end': 55967909, 'start': 55967909},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.59914,\n",
       "     'fitcons_score': 0.651492},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.49028,\n",
       "     'fitcons_score': 0.660085},\n",
       "    'interpro_domain': 'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain',\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['E57*', 'E57*', 'E57*', '.'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'without_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'D'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 1.0, 'mammalian_rankscore': 0.88762}},\n",
       "    'phylo': {'p100way': {'vertebrate': 9.985,\n",
       "      'vertebrate_rankscore': 0.99176},\n",
       "     'p20way': {'mammalian': 0.978, 'mammalian_rankscore': 0.59618}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'G',\n",
       "    'rsid': 'rs757201758',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 17.0098,\n",
       "     'logodds_rankscore': 0.86213,\n",
       "     'pi': {'a': 0.0, 'c': 0.0, 'g': 1.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'dbsnp': {'allele_origin': 'unspecified',\n",
       "    'alleles': [{'allele': 'G'}, {'allele': 'T'}],\n",
       "    'alt': 'T',\n",
       "    'chrom': '12',\n",
       "    'class': 'SNV',\n",
       "    'dbsnp_build': 144,\n",
       "    'flags': ['ASP', 'INT', 'NSN', 'R5', 'REF'],\n",
       "    'gene': [{'geneid': '1017', 'symbol': 'CDK2'},\n",
       "     {'geneid': '6490', 'symbol': 'PMEL'}],\n",
       "    'hg19': {'end': 56361693, 'start': 56361693},\n",
       "    'ref': 'G',\n",
       "    'rsid': 'rs757201758',\n",
       "    'validated': False,\n",
       "    'var_subtype': 'tv',\n",
       "    'vartype': 'snp'},\n",
       "   'exac': {'ac': {'ac': 1,\n",
       "     'ac_adj': 1,\n",
       "     'ac_afr': 0,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 0,\n",
       "     'ac_female': 1,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 1,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 0,\n",
       "     'ac_nfe': 1,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 8.236e-06,\n",
       "    'alleles': 'T',\n",
       "    'alt': 'T',\n",
       "    'an': {'an': 121412,\n",
       "     'an_adj': 121396,\n",
       "     'an_afr': 10406,\n",
       "     'an_amr': 11572,\n",
       "     'an_eas': 8654,\n",
       "     'an_female': 54114,\n",
       "     'an_fin': 6614,\n",
       "     'an_male': 67282,\n",
       "     'an_nfe': 66732,\n",
       "     'an_oth': 906,\n",
       "     'an_sas': 16512},\n",
       "    'baseqranksum': -2.257,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.394,\n",
       "    'culprit': 'MQRankSum',\n",
       "    'fs': 2.112,\n",
       "    'het': {'het_afr': 0,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 0,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 1,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': 0.0004,\n",
       "    'mq': {'mq': 59.71, 'mq0': 0, 'mqranksum': -0.856},\n",
       "    'ncc': 7,\n",
       "    'pos': 56361693,\n",
       "    'qd': 11.01,\n",
       "    'readposranksum': 0.802,\n",
       "    'ref': 'G',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 0.69},\n",
       "   'exac_nontcga': {'ac': {'ac': 1,\n",
       "     'ac_adj': 1,\n",
       "     'ac_afr': 0,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 0,\n",
       "     'ac_female': 1,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 1,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 0,\n",
       "     'ac_nfe': 1,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 9.415e-06,\n",
       "    'alleles': 'T',\n",
       "    'alt': 'T',\n",
       "    'an': {'an': 106210,\n",
       "     'an_adj': 106196,\n",
       "     'an_afr': 9066,\n",
       "     'an_amr': 11210,\n",
       "     'an_eas': 7866,\n",
       "     'an_female': 45866,\n",
       "     'an_fin': 6614,\n",
       "     'an_male': 60330,\n",
       "     'an_nfe': 54338,\n",
       "     'an_oth': 694,\n",
       "     'an_sas': 16408},\n",
       "    'baseqranksum': -2.257,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.394,\n",
       "    'culprit': 'MQRankSum',\n",
       "    'fs': 2.112,\n",
       "    'het': {'het_afr': 0,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 0,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 1,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': 0.0004,\n",
       "    'mq': {'mq': 59.71, 'mq0': 0, 'mqranksum': -0.856},\n",
       "    'ncc': 7,\n",
       "    'pos': 56361693,\n",
       "    'qd': 11.01,\n",
       "    'readposranksum': 0.802,\n",
       "    'ref': 'G',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 0.69},\n",
       "   'hg19': {'end': 56361693, 'start': 56361693},\n",
       "   'observed': True,\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '409'},\n",
       "      'cds': {'length': '897', 'position': '169'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.169G>T',\n",
       "      'hgvs_p': 'p.Glu57*',\n",
       "      'protein': {'length': '298', 'position': '57'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '2',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '409'},\n",
       "      'cds': {'length': '795', 'position': '169'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.169G>T',\n",
       "      'hgvs_p': 'p.Glu57*',\n",
       "      'protein': {'length': '264', 'position': '57'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '2',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '1847',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52C>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '1847',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52C>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '1197',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87C>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.117-140G>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '1',\n",
       "      'total': '4',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '0.67'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '0.67'}},\n",
       "   'vcf': {'alt': 'T', 'position': '56361693', 'ref': 'G'}},\n",
       "  {'_id': 'chr12:g.56362574C>T',\n",
       "   '_score': 7.2263837,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'C',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 250,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.071,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.016,\n",
       "     'tssaflnk': 0.008,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.15,\n",
       "     'txflnk': 0.039,\n",
       "     'txwk': 0.646,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.03,\n",
       "    'dna': {'helt': 1.21, 'mgw': 0.62, 'prot': -2.97, 'roll': 1.34},\n",
       "    'dst2splice': 13,\n",
       "    'dst2spltype': 'ACCEPTOR',\n",
       "    'encode': {'exp': 193.32,\n",
       "     'h3k27ac': 7.16,\n",
       "     'h3k4me1': 14.12,\n",
       "     'h3k4me3': 19.96,\n",
       "     'nucleo': 2.2},\n",
       "    'exon': '4/7',\n",
       "    'fitcons': 0.726386,\n",
       "    'gc': 0.5,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 568,\n",
       "      'cds_pos': 328,\n",
       "      'rel_cdna_pos': 0.25,\n",
       "      'rel_cds_pos': 0.37},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 110, 'rel_prot_pos': 0.37}},\n",
       "    'gerp': {'n': 4.53, 'rs': 602.1, 'rs_pval': 2.56976e-59, 's': 4.53},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'FALSE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 479,\n",
       "    'min_dist_tss': 1328,\n",
       "    'mutindex': -26,\n",
       "    'naa': '*',\n",
       "    'oaa': 'Q',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.985, 'vertebrate': 1.0},\n",
       "    'phred': 39,\n",
       "    'phylop': {'mammalian': 2.522, 'primate': 0.457, 'vertebrate': 5.459},\n",
       "    'pos': 56362574,\n",
       "    'rawscore': 12.632847,\n",
       "    'ref': 'C',\n",
       "    'segway': 'GE1',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 1,\n",
       "     'pos': ['110', '110', '84', '110'],\n",
       "     'ref': 'Q',\n",
       "     'refcodon': 'CAG'},\n",
       "    'alt': 'T',\n",
       "    'ancestral_allele': 'C',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.89712,\n",
       "     'score': 0.9981459634050125},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 17.64166,\n",
       "     'raw': 1.10312033626684},\n",
       "    'eigen-pc': {'phred': 15.25602,\n",
       "     'raw': 0.933325513156562,\n",
       "     'raw_rankscore': 0.96387},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFBI',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.77402,\n",
       "     'coding_score': 0.97845},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.99999999999996},\n",
       "    'gerp++': {'nr': 4.53, 'rs': 4.53, 'rs_rankscore': 0.54846},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8784,\n",
       "     'fitcons_score': 0.724815},\n",
       "    'hg18': {'end': 54648841, 'start': 54648841},\n",
       "    'hg19': {'end': 56362574, 'start': 56362574},\n",
       "    'hg38': {'end': 55968790, 'start': 55968790},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.9842,\n",
       "     'fitcons_score': 0.735409},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8535,\n",
       "     'fitcons_score': 0.722319},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['Q110*', 'Q84*', 'Q110*', 'Q110*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 1.0, 'mammalian_rankscore': 0.88762}},\n",
       "    'phylo': {'p100way': {'vertebrate': 7.866,\n",
       "      'vertebrate_rankscore': 0.85362},\n",
       "     'p20way': {'mammalian': 0.886, 'mammalian_rankscore': 0.38909}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'C',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 16.5654,\n",
       "     'logodds_rankscore': 0.84367,\n",
       "     'pi': {'a': 0.0, 'c': 1.0, 'g': 0.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56362574, 'start': 56362574},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '568'},\n",
       "      'cds': {'length': '897', 'position': '328'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.328C>T',\n",
       "      'hgvs_p': 'p.Gln110*',\n",
       "      'protein': {'length': '298', 'position': '110'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '4',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '490'},\n",
       "      'cds': {'length': '717', 'position': '250'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.250C>T',\n",
       "      'hgvs_p': 'p.Gln84*',\n",
       "      'protein': {'length': '238', 'position': '84'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '568'},\n",
       "      'cds': {'length': '795', 'position': '328'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.328C>T',\n",
       "      'hgvs_p': 'p.Gln110*',\n",
       "      'protein': {'length': '264', 'position': '110'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '4',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2728',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2728',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2078',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'T', 'position': '56362574', 'ref': 'C'}},\n",
       "  {'_id': 'chr12:g.56361903A>T',\n",
       "   '_score': 7.2185206,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'A',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 251,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.055,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.047,\n",
       "     'tssaflnk': 0.094,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.0,\n",
       "     'txflnk': 0.11,\n",
       "     'txwk': 0.457,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.0,\n",
       "    'dna': {'helt': -3.16, 'mgw': 2.28, 'prot': 4.14, 'roll': 13.24},\n",
       "    'encode': {'exp': 696.29,\n",
       "     'h3k27ac': 10.16,\n",
       "     'h3k4me1': 19.92,\n",
       "     'h3k4me3': 29.88,\n",
       "     'nucleo': 1.1},\n",
       "    'exon': '3/7',\n",
       "    'fitcons': 0.623825,\n",
       "    'gc': 0.44,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 505,\n",
       "      'cds_pos': 265,\n",
       "      'rel_cdna_pos': 0.22,\n",
       "      'rel_cds_pos': 0.3},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 89, 'rel_prot_pos': 0.3}},\n",
       "    'gerp': {'n': 5.26, 'rs': 504.2, 'rs_pval': 6.27736e-64, 's': 5.26},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 193,\n",
       "    'min_dist_tss': 645,\n",
       "    'mutindex': -43,\n",
       "    'naa': '*',\n",
       "    'oaa': 'K',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.995, 'vertebrate': 1.0},\n",
       "    'phred': 40,\n",
       "    'phylop': {'mammalian': 2.122, 'primate': 0.398, 'vertebrate': 3.1},\n",
       "    'pos': 56361903,\n",
       "    'rawscore': 12.734282,\n",
       "    'ref': 'A',\n",
       "    'segway': 'E/GM',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 1,\n",
       "     'pos': ['89', '89', '63', '89'],\n",
       "     'ref': 'K',\n",
       "     'refcodon': 'AAA'},\n",
       "    'alt': 'T',\n",
       "    'ancestral_allele': 'A',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.72936,\n",
       "     'score': 0.9958366694956521},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 9.623577,\n",
       "     'raw': 0.840452296876286},\n",
       "    'eigen-pc': {'phred': 8.147482,\n",
       "     'raw': 0.720175393750942,\n",
       "     'raw_rankscore': 0.83692},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFB',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.75326,\n",
       "     'coding_score': 0.97552},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999974230988},\n",
       "    'gerp++': {'nr': 5.26, 'rs': 5.26, 'rs_rankscore': 0.73376},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.61804,\n",
       "     'fitcons_score': 0.685571},\n",
       "    'hg18': {'end': 54648170, 'start': 54648170},\n",
       "    'hg19': {'end': 56361903, 'start': 56361903},\n",
       "    'hg38': {'end': 55968119, 'start': 55968119},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.59914,\n",
       "     'fitcons_score': 0.651492},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.49028,\n",
       "     'fitcons_score': 0.660085},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.049683,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['K89*', 'K63*', 'K89*', 'K89*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 1.0, 'mammalian_rankscore': 0.88762}},\n",
       "    'phylo': {'p100way': {'vertebrate': 9.248, 'vertebrate_rankscore': 0.9458},\n",
       "     'p20way': {'mammalian': 1.118, 'mammalian_rankscore': 0.88034}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'A',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 14.4595,\n",
       "     'logodds_rankscore': 0.66874,\n",
       "     'pi': {'a': 1.0, 'c': 0.0, 'g': 0.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56361903, 'start': 56361903},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '505'},\n",
       "      'cds': {'length': '897', 'position': '265'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.265A>T',\n",
       "      'hgvs_p': 'p.Lys89*',\n",
       "      'protein': {'length': '298', 'position': '89'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '427'},\n",
       "      'cds': {'length': '717', 'position': '187'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.187A>T',\n",
       "      'hgvs_p': 'p.Lys63*',\n",
       "      'protein': {'length': '238', 'position': '63'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '2',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '505'},\n",
       "      'cds': {'length': '795', 'position': '265'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.265A>T',\n",
       "      'hgvs_p': 'p.Lys89*',\n",
       "      'protein': {'length': '264', 'position': '89'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2057',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2057',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '1407',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'T', 'position': '56361903', 'ref': 'A'}},\n",
       "  {'_id': 'chr12:g.56361869C>A',\n",
       "   '_score': 7.1632676,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'A',\n",
       "    'anc': 'C',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 251,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.055,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.047,\n",
       "     'tssaflnk': 0.094,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.0,\n",
       "     'txflnk': 0.11,\n",
       "     'txwk': 0.457,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.0,\n",
       "    'dna': {'helt': 1.06, 'mgw': -0.35, 'prot': -3.67, 'roll': -2.25},\n",
       "    'encode': {'exp': 683.87,\n",
       "     'h3k27ac': 11.52,\n",
       "     'h3k4me1': 21.44,\n",
       "     'h3k4me3': 26.48,\n",
       "     'nucleo': 1.2},\n",
       "    'exon': '3/7',\n",
       "    'fitcons': 0.623825,\n",
       "    'gc': 0.44,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 471,\n",
       "      'cds_pos': 231,\n",
       "      'rel_cdna_pos': 0.21,\n",
       "      'rel_cds_pos': 0.26},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 77, 'rel_prot_pos': 0.26}},\n",
       "    'gerp': {'n': 5.26, 'rs': 504.2, 'rs_pval': 6.27736e-64, 's': 4.36},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 227,\n",
       "    'min_dist_tss': 611,\n",
       "    'mutindex': -2,\n",
       "    'naa': '*',\n",
       "    'oaa': 'Y',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.997, 'vertebrate': 1.0},\n",
       "    'phred': 36,\n",
       "    'phylop': {'mammalian': 1.329, 'primate': 0.486, 'vertebrate': 3.117},\n",
       "    'pos': 56361869,\n",
       "    'rawscore': 10.033274,\n",
       "    'ref': 'C',\n",
       "    'segway': 'E/GM',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 2,\n",
       "     'codonpos': 3,\n",
       "     'pos': ['77', '77', '51', '77'],\n",
       "     'ref': 'Y',\n",
       "     'refcodon': 'TAC'},\n",
       "    'alt': 'A',\n",
       "    'ancestral_allele': 'C',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.82906,\n",
       "     'score': 0.9973468632930617},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 6.40465,\n",
       "     'raw': 0.644141070663874},\n",
       "    'eigen-pc': {'phred': 5.174308,\n",
       "     'raw': 0.501693885127163,\n",
       "     'raw_rankscore': 0.6821},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFB',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.68415,\n",
       "     'coding_score': 0.96333},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999937496681},\n",
       "    'gerp++': {'nr': 5.26, 'rs': 4.36, 'rs_rankscore': 0.51475},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.61804,\n",
       "     'fitcons_score': 0.685571},\n",
       "    'hg18': {'end': 54648136, 'start': 54648136},\n",
       "    'hg19': {'end': 56361869, 'start': 56361869},\n",
       "    'hg38': {'end': 55968085, 'start': 55968085},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.59914,\n",
       "     'fitcons_score': 0.651492},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.49028,\n",
       "     'fitcons_score': 0.660085},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['Y77*', 'Y51*', 'Y77*', 'Y77*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 0.999, 'mammalian_rankscore': 0.74975}},\n",
       "    'phylo': {'p100way': {'vertebrate': 1.024,\n",
       "      'vertebrate_rankscore': 0.29476},\n",
       "     'p20way': {'mammalian': 0.038, 'mammalian_rankscore': 0.15345}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'C',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 12.2421,\n",
       "     'logodds_rankscore': 0.53681,\n",
       "     'pi': {'a': 0.0, 'c': 0.9144, 'g': 0.0, 't': 0.0856}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56361869, 'start': 56361869},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '471'},\n",
       "      'cds': {'length': '897', 'position': '231'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.231C>A',\n",
       "      'hgvs_p': 'p.Tyr77*',\n",
       "      'protein': {'length': '298', 'position': '77'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '393'},\n",
       "      'cds': {'length': '717', 'position': '153'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.153C>A',\n",
       "      'hgvs_p': 'p.Tyr51*',\n",
       "      'protein': {'length': '238', 'position': '51'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '2',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '471'},\n",
       "      'cds': {'length': '795', 'position': '231'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.231C>A',\n",
       "      'hgvs_p': 'p.Tyr77*',\n",
       "      'protein': {'length': '264', 'position': '77'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2023',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52G>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2023',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52G>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '1373',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87G>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'A', 'position': '56361869', 'ref': 'C'}},\n",
       "  {'_id': 'chr12:g.56362567T>G',\n",
       "   '_score': 7.1632676,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'G',\n",
       "    'anc': 'T',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 250,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.071,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.016,\n",
       "     'tssaflnk': 0.008,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.15,\n",
       "     'txflnk': 0.039,\n",
       "     'txwk': 0.646,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.03,\n",
       "    'dna': {'helt': -1.67, 'mgw': 0.25, 'prot': 2.06, 'roll': 0.17},\n",
       "    'dst2splice': 6,\n",
       "    'dst2spltype': 'ACCEPTOR',\n",
       "    'encode': {'exp': 154.41,\n",
       "     'h3k27ac': 7.16,\n",
       "     'h3k4me1': 14.12,\n",
       "     'h3k4me3': 19.96,\n",
       "     'nucleo': 2.8},\n",
       "    'exon': '4/7',\n",
       "    'fitcons': 0.726386,\n",
       "    'gc': 0.52,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 561,\n",
       "      'cds_pos': 321,\n",
       "      'rel_cdna_pos': 0.24,\n",
       "      'rel_cds_pos': 0.36},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 107, 'rel_prot_pos': 0.36}},\n",
       "    'gerp': {'n': 4.53, 'rs': 602.1, 'rs_pval': 2.56976e-59, 's': 2.07},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 472,\n",
       "    'min_dist_tss': 1321,\n",
       "    'mutindex': -13,\n",
       "    'naa': '*',\n",
       "    'oaa': 'Y',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.961, 'vertebrate': 1.0},\n",
       "    'phred': 36,\n",
       "    'phylop': {'mammalian': 0.808, 'primate': 0.375, 'vertebrate': 0.429},\n",
       "    'pos': 56362567,\n",
       "    'rawscore': 9.700482,\n",
       "    'ref': 'T',\n",
       "    'segway': 'GE1',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 2,\n",
       "     'codonpos': 3,\n",
       "     'pos': ['107', '107', '81', '107'],\n",
       "     'ref': 'Y',\n",
       "     'refcodon': 'TAT'},\n",
       "    'alt': 'G',\n",
       "    'ancestral_allele': 'T',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.77255,\n",
       "     'score': 0.996532737764271},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 5.131892,\n",
       "     'raw': 0.512233895100652},\n",
       "    'eigen-pc': {'phred': 3.893138,\n",
       "     'raw': 0.329494801375175,\n",
       "     'raw_rankscore': 0.57571},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFBI',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.48831,\n",
       "     'coding_score': 0.8894},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.42663, 'score': 0.999765088166355},\n",
       "    'gerp++': {'nr': 4.53, 'rs': 2.07, 'rs_rankscore': 0.25882},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8784,\n",
       "     'fitcons_score': 0.724815},\n",
       "    'hg18': {'end': 54648834, 'start': 54648834},\n",
       "    'hg19': {'end': 56362567, 'start': 56362567},\n",
       "    'hg38': {'end': 55968783, 'start': 55968783},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.9842,\n",
       "     'fitcons_score': 0.735409},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8535,\n",
       "     'fitcons_score': 0.722319},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['Y107*', 'Y81*', 'Y107*', 'Y107*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 1.0, 'mammalian_rankscore': 0.88762}},\n",
       "    'phylo': {'p100way': {'vertebrate': 1.715, 'vertebrate_rankscore': 0.3736},\n",
       "     'p20way': {'mammalian': 1.004, 'mammalian_rankscore': 0.6225}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'T',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 8.4104,\n",
       "     'logodds_rankscore': 0.31622,\n",
       "     'pi': {'a': 0.0, 'c': 0.1901, 'g': 0.0, 't': 0.8099}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56362567, 'start': 56362567},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '561'},\n",
       "      'cds': {'length': '897', 'position': '321'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.321T>G',\n",
       "      'hgvs_p': 'p.Tyr107*',\n",
       "      'protein': {'length': '298', 'position': '107'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '4',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '483'},\n",
       "      'cds': {'length': '717', 'position': '243'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.243T>G',\n",
       "      'hgvs_p': 'p.Tyr81*',\n",
       "      'protein': {'length': '238', 'position': '81'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '561'},\n",
       "      'cds': {'length': '795', 'position': '321'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.321T>G',\n",
       "      'hgvs_p': 'p.Tyr107*',\n",
       "      'protein': {'length': '264', 'position': '107'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '4',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2721',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52A>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2721',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52A>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2071',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87A>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'G', 'position': '56362567', 'ref': 'T'}},\n",
       "  {'_id': 'chr12:g.56361900A>T',\n",
       "   '_score': 7.123927,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'A',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 251,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.055,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.047,\n",
       "     'tssaflnk': 0.094,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.0,\n",
       "     'txflnk': 0.11,\n",
       "     'txwk': 0.457,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.0,\n",
       "    'dna': {'helt': -0.9, 'mgw': 0.88, 'prot': 3.28, 'roll': 7.71},\n",
       "    'encode': {'exp': 708.71,\n",
       "     'h3k27ac': 9.44,\n",
       "     'h3k4me1': 22.4,\n",
       "     'h3k4me3': 24.48,\n",
       "     'nucleo': 1.1},\n",
       "    'exon': '3/7',\n",
       "    'fitcons': 0.623825,\n",
       "    'gc': 0.44,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 502,\n",
       "      'cds_pos': 262,\n",
       "      'rel_cdna_pos': 0.22,\n",
       "      'rel_cds_pos': 0.29},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 88, 'rel_prot_pos': 0.3}},\n",
       "    'gerp': {'n': 5.26, 'rs': 504.2, 'rs_pval': 6.27736e-64, 's': 5.26},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 196,\n",
       "    'min_dist_tss': 642,\n",
       "    'mutindex': -36,\n",
       "    'naa': '*',\n",
       "    'oaa': 'K',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.994, 'vertebrate': 1.0},\n",
       "    'phred': 38,\n",
       "    'phylop': {'mammalian': 2.122, 'primate': 0.398, 'vertebrate': 4.77},\n",
       "    'pos': 56361900,\n",
       "    'rawscore': 11.826591,\n",
       "    'ref': 'A',\n",
       "    'segway': 'E/GM',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 1,\n",
       "     'pos': ['88', '88', '62', '88'],\n",
       "     'ref': 'K',\n",
       "     'refcodon': 'AAG'},\n",
       "    'alt': 'T',\n",
       "    'ancestral_allele': 'A',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.76439,\n",
       "     'score': 0.9964034676927931},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 9.620236,\n",
       "     'raw': 0.840301643323044},\n",
       "    'eigen-pc': {'phred': 8.143378,\n",
       "     'raw': 0.719973840873594,\n",
       "     'raw_rankscore': 0.8367},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFB',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.83996,\n",
       "     'coding_score': 0.98561},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999999997431},\n",
       "    'gerp++': {'nr': 5.26, 'rs': 5.26, 'rs_rankscore': 0.73376},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.61804,\n",
       "     'fitcons_score': 0.685571},\n",
       "    'hg18': {'end': 54648167, 'start': 54648167},\n",
       "    'hg19': {'end': 56361900, 'start': 56361900},\n",
       "    'hg38': {'end': 55968116, 'start': 55968116},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.59914,\n",
       "     'fitcons_score': 0.651492},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.49028,\n",
       "     'fitcons_score': 0.660085},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['K88*', 'K62*', 'K88*', 'K88*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 0.998, 'mammalian_rankscore': 0.69722}},\n",
       "    'phylo': {'p100way': {'vertebrate': 9.248, 'vertebrate_rankscore': 0.9458},\n",
       "     'p20way': {'mammalian': 1.118, 'mammalian_rankscore': 0.88034}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'A',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 14.4595,\n",
       "     'logodds_rankscore': 0.66874,\n",
       "     'pi': {'a': 1.0, 'c': 0.0, 'g': 0.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56361900, 'start': 56361900},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '502'},\n",
       "      'cds': {'length': '897', 'position': '262'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.262A>T',\n",
       "      'hgvs_p': 'p.Lys88*',\n",
       "      'protein': {'length': '298', 'position': '88'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '424'},\n",
       "      'cds': {'length': '717', 'position': '184'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.184A>T',\n",
       "      'hgvs_p': 'p.Lys62*',\n",
       "      'protein': {'length': '238', 'position': '62'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '2',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '502'},\n",
       "      'cds': {'length': '795', 'position': '262'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.262A>T',\n",
       "      'hgvs_p': 'p.Lys88*',\n",
       "      'protein': {'length': '264', 'position': '88'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2054',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2054',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '1404',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'T', 'position': '56361900', 'ref': 'A'}},\n",
       "  {'_id': 'chr12:g.56363304A>T',\n",
       "   '_score': 7.10932,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'A',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 250,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.063,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.0,\n",
       "     'tssaflnk': 0.0,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.268,\n",
       "     'txflnk': 0.024,\n",
       "     'txwk': 0.583,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.04,\n",
       "    'dna': {'helt': -1.69, 'mgw': 1.56, 'prot': 1.27, 'roll': 8.9},\n",
       "    'encode': {'exp': 278.18,\n",
       "     'h3k27ac': 7.28,\n",
       "     'h3k4me1': 7.08,\n",
       "     'h3k4me3': 7.88,\n",
       "     'nucleo': 0.6},\n",
       "    'exon': '5/7',\n",
       "    'fitcons': 0.722197,\n",
       "    'gc': 0.58,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 772,\n",
       "      'cds_pos': 532,\n",
       "      'rel_cdna_pos': 0.34,\n",
       "      'rel_cds_pos': 0.59},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 178, 'rel_prot_pos': 0.6}},\n",
       "    'gerp': {'n': 5.44, 'rs': 457.7, 'rs_pval': 1.6043e-69, 's': 5.44},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 364,\n",
       "    'min_dist_tss': 2058,\n",
       "    'mutindex': -44,\n",
       "    'naa': '*',\n",
       "    'oaa': 'K',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.977, 'vertebrate': 1.0},\n",
       "    'phred': 41,\n",
       "    'phylop': {'mammalian': 2.365, 'primate': 0.525, 'vertebrate': 3.25},\n",
       "    'pos': 56363304,\n",
       "    'rawscore': 13.329179,\n",
       "    'ref': 'A',\n",
       "    'segway': 'TF2',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 1,\n",
       "     'pos': 178,\n",
       "     'ref': 'K',\n",
       "     'refcodon': 'AAA'},\n",
       "    'alt': 'T',\n",
       "    'ancestral_allele': 'A',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.74665,\n",
       "     'score': 0.9961165327581305},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 13.05126,\n",
       "     'raw': 0.971550766170071},\n",
       "    'eigen-pc': {'phred': 11.7538,\n",
       "     'raw': 0.849496518123564,\n",
       "     'raw_rankscore': 0.92577},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970', 'ENSP00000452514'],\n",
       "     'transcriptid': ['ENST00000266970', 'ENST00000553376']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFDBI',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.66943,\n",
       "     'coding_score': 0.96018},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999989307069},\n",
       "    'gerp++': {'nr': 5.44, 'rs': 5.44, 'rs_rankscore': 0.7927},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.82299,\n",
       "     'fitcons_score': 0.723133},\n",
       "    'hg18': {'end': 54649571, 'start': 54649571},\n",
       "    'hg19': {'end': 56363304, 'start': 56363304},\n",
       "    'hg38': {'end': 55969520, 'start': 55969520},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.83238,\n",
       "     'fitcons_score': 0.714379},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.83035,\n",
       "     'fitcons_score': 0.719381},\n",
       "    'interpro_domain': 'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain',\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['K178*', 'K178*', '.', '.'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'without_aae', 'without_aae'],\n",
       "     'pred': ['A', 'A', 'D', 'D'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 0.999, 'mammalian_rankscore': 0.74975}},\n",
       "    'phylo': {'p100way': {'vertebrate': 7.335,\n",
       "      'vertebrate_rankscore': 0.78321},\n",
       "     'p20way': {'mammalian': 1.162, 'mammalian_rankscore': 0.90104}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'A',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 15.1728,\n",
       "     'logodds_rankscore': 0.72451,\n",
       "     'pi': {'a': 1.0, 'c': 0.0, 'g': 0.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56363304, 'start': 56363304},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '772'},\n",
       "      'cds': {'length': '897', 'position': '532'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.532A>T',\n",
       "      'hgvs_p': 'p.Lys178*',\n",
       "      'protein': {'length': '298', 'position': '178'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '5',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '3458',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4393',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252036.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-181A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '3458',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2808',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4491',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252037.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4491',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_002868.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.408+572A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '3',\n",
       "      'total': '4',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.486+572A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '4',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '0.33'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '0.33'}},\n",
       "   'vcf': {'alt': 'T', 'position': '56363304', 'ref': 'A'}},\n",
       "  {'_id': 'chr12:g.56361861A>T',\n",
       "   '_score': 7.091785,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'A',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 251,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.055,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.047,\n",
       "     'tssaflnk': 0.094,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.0,\n",
       "     'txflnk': 0.11,\n",
       "     'txwk': 0.457,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.0,\n",
       "    'dna': {'helt': -3.16, 'mgw': 2.28, 'prot': 4.14, 'roll': 13.24},\n",
       "    'encode': {'exp': 678.9,\n",
       "     'h3k27ac': 11.52,\n",
       "     'h3k4me1': 21.44,\n",
       "     'h3k4me3': 26.48,\n",
       "     'nucleo': 1.3},\n",
       "    'exon': '3/7',\n",
       "    'fitcons': 0.623825,\n",
       "    'gc': 0.42,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 463,\n",
       "      'cds_pos': 223,\n",
       "      'rel_cdna_pos': 0.2,\n",
       "      'rel_cds_pos': 0.25},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 75, 'rel_prot_pos': 0.25}},\n",
       "    'gerp': {'n': 5.26, 'rs': 504.2, 'rs_pval': 6.27736e-64, 's': 5.26},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 235,\n",
       "    'min_dist_tss': 603,\n",
       "    'mutindex': -21,\n",
       "    'naa': '*',\n",
       "    'oaa': 'K',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.996, 'vertebrate': 1.0},\n",
       "    'phred': 40,\n",
       "    'phylop': {'mammalian': 2.122, 'primate': 0.398, 'vertebrate': 3.1},\n",
       "    'pos': 56361861,\n",
       "    'rawscore': 12.755175,\n",
       "    'ref': 'A',\n",
       "    'segway': 'E/GM',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 1,\n",
       "     'pos': ['75', '75', '49', '75'],\n",
       "     'ref': 'K',\n",
       "     'refcodon': 'AAA'},\n",
       "    'alt': 'T',\n",
       "    'ancestral_allele': 'A',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.7215,\n",
       "     'score': 0.9957055916952746},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 10.4152,\n",
       "     'raw': 0.874627814114319},\n",
       "    'eigen-pc': {'phred': 8.938035,\n",
       "     'raw': 0.755795664242958,\n",
       "     'raw_rankscore': 0.86307},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFB',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.83661,\n",
       "     'coding_score': 0.98531},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999974230988},\n",
       "    'gerp++': {'nr': 5.26, 'rs': 5.26, 'rs_rankscore': 0.73376},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.61804,\n",
       "     'fitcons_score': 0.685571},\n",
       "    'hg18': {'end': 54648128, 'start': 54648128},\n",
       "    'hg19': {'end': 56361861, 'start': 56361861},\n",
       "    'hg38': {'end': 55968077, 'start': 55968077},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.59914,\n",
       "     'fitcons_score': 0.651492},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.49028,\n",
       "     'fitcons_score': 0.660085},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['K75*', 'K49*', 'K75*', 'K75*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 1.0, 'mammalian_rankscore': 0.88762}},\n",
       "    'phylo': {'p100way': {'vertebrate': 7.456,\n",
       "      'vertebrate_rankscore': 0.79641},\n",
       "     'p20way': {'mammalian': 1.118, 'mammalian_rankscore': 0.88034}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'A',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 14.4595,\n",
       "     'logodds_rankscore': 0.66874,\n",
       "     'pi': {'a': 1.0, 'c': 0.0, 'g': 0.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56361861, 'start': 56361861},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '463'},\n",
       "      'cds': {'length': '897', 'position': '223'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.223A>T',\n",
       "      'hgvs_p': 'p.Lys75*',\n",
       "      'protein': {'length': '298', 'position': '75'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '385'},\n",
       "      'cds': {'length': '717', 'position': '145'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.145A>T',\n",
       "      'hgvs_p': 'p.Lys49*',\n",
       "      'protein': {'length': '238', 'position': '49'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '2',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '463'},\n",
       "      'cds': {'length': '795', 'position': '223'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.223A>T',\n",
       "      'hgvs_p': 'p.Lys75*',\n",
       "      'protein': {'length': '264', 'position': '75'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2015',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2015',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '1365',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'T', 'position': '56361861', 'ref': 'A'}},\n",
       "  {'_id': 'chr12:g.56364963A>T',\n",
       "   '_score': 7.091785,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'A',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 241,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.094,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.0,\n",
       "     'tssaflnk': 0.0,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.331,\n",
       "     'txflnk': 0.024,\n",
       "     'txwk': 0.496,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.05,\n",
       "    'dna': {'helt': -0.77, 'mgw': 0.97, 'prot': 2.54, 'roll': 8.9},\n",
       "    'encode': {'exp': 822.54,\n",
       "     'h3k27ac': 9.84,\n",
       "     'h3k4me1': 10.12,\n",
       "     'h3k4me3': 6.6,\n",
       "     'nucleo': 0.7},\n",
       "    'exon': '6/7',\n",
       "    'fitcons': 0.701516,\n",
       "    'gc': 0.53,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 964,\n",
       "      'cds_pos': 724,\n",
       "      'rel_cdna_pos': 0.42,\n",
       "      'rel_cds_pos': 0.81},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 242, 'rel_prot_pos': 0.81}},\n",
       "    'gerp': {'n': 5.12, 'rs': 2645.2, 'rs_pval': 9.82059e-310, 's': 5.12},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 80,\n",
       "    'min_dist_tss': 59,\n",
       "    'mutindex': -32,\n",
       "    'naa': '*',\n",
       "    'oaa': 'K',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.868, 'vertebrate': 1.0},\n",
       "    'phred': 38,\n",
       "    'phylop': {'mammalian': 2.066, 'primate': 0.478, 'vertebrate': 4.714},\n",
       "    'pos': 56364963,\n",
       "    'rawscore': 12.185485,\n",
       "    'ref': 'A',\n",
       "    'segway': 'GE1',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 1,\n",
       "     'pos': ['242', '290', '182', '208'],\n",
       "     'ref': 'K',\n",
       "     'refcodon': 'AAG'},\n",
       "    'alt': 'T',\n",
       "    'ancestral_allele': 'A',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.72395,\n",
       "     'score': 0.9957443054637624},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 8.504234,\n",
       "     'raw': 0.785569528099217},\n",
       "    'eigen-pc': {'phred': 6.924379,\n",
       "     'raw': 0.650882296532907,\n",
       "     'raw_rankscore': 0.78559},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFBCIJ',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.3842,\n",
       "     'coding_score': 0.78455},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999999999909},\n",
       "    'gerp++': {'nr': 5.12, 'rs': 5.12, 'rs_rankscore': 0.69338},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8784,\n",
       "     'fitcons_score': 0.724815},\n",
       "    'hg18': {'end': 54651230, 'start': 54651230},\n",
       "    'hg19': {'end': 56364963, 'start': 56364963},\n",
       "    'hg38': {'end': 55971179, 'start': 55971179},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.83238,\n",
       "     'fitcons_score': 0.714379},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.72963,\n",
       "     'fitcons_score': 0.706548},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['K208*', 'K182*', 'K290*', 'K242*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 1.0, 'mammalian_rankscore': 0.88762}},\n",
       "    'phylo': {'p100way': {'vertebrate': 4.968,\n",
       "      'vertebrate_rankscore': 0.63211},\n",
       "     'p20way': {'mammalian': 1.118, 'mammalian_rankscore': 0.88034}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'A',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 14.2119,\n",
       "     'logodds_rankscore': 0.65164,\n",
       "     'pi': {'a': 1.0, 'c': 0.0, 'g': 0.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56364963, 'start': 56364963},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '964'},\n",
       "      'cds': {'length': '897', 'position': '724'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.724A>T',\n",
       "      'hgvs_p': 'p.Lys242*',\n",
       "      'protein': {'length': '298', 'position': '242'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '6',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '784'},\n",
       "      'cds': {'length': '717', 'position': '544'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.544A>T',\n",
       "      'hgvs_p': 'p.Lys182*',\n",
       "      'protein': {'length': '238', 'position': '182'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '4',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '862'},\n",
       "      'cds': {'length': '795', 'position': '622'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.622A>T',\n",
       "      'hgvs_p': 'p.Lys208*',\n",
       "      'protein': {'length': '264', 'position': '208'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '5',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2734',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252036.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-181A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4467',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2832',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252037.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2832',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_002868.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'T', 'position': '56364963', 'ref': 'A'}},\n",
       "  {'_id': 'chr12:g.56364920G>A',\n",
       "   '_score': 7.0576696,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'A',\n",
       "    'anc': 'G',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 241,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.094,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.0,\n",
       "     'tssaflnk': 0.0,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.331,\n",
       "     'txflnk': 0.024,\n",
       "     'txwk': 0.496,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.04,\n",
       "    'dna': {'helt': -1.87, 'mgw': 0.07, 'prot': 1.32, 'roll': -0.88},\n",
       "    'encode': {'exp': 602.32,\n",
       "     'h3k27ac': 6.56,\n",
       "     'h3k4me1': 11.0,\n",
       "     'h3k4me3': 6.0,\n",
       "     'nucleo': 2.9},\n",
       "    'exon': '6/7',\n",
       "    'fitcons': 0.701516,\n",
       "    'gc': 0.5,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 921,\n",
       "      'cds_pos': 681,\n",
       "      'rel_cdna_pos': 0.4,\n",
       "      'rel_cds_pos': 0.76},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 227, 'rel_prot_pos': 0.76}},\n",
       "    'gerp': {'n': 5.22, 'rs': 2645.2, 'rs_pval': 9.82059e-310, 's': 5.22},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'FALSE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 37,\n",
       "    'min_dist_tss': 16,\n",
       "    'mutindex': -27,\n",
       "    'naa': '*',\n",
       "    'oaa': 'W',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.782, 'vertebrate': 1.0},\n",
       "    'phred': 37,\n",
       "    'phylop': {'mammalian': 2.613, 'primate': 0.587, 'vertebrate': 5.764},\n",
       "    'pos': 56364920,\n",
       "    'rawscore': 11.158933,\n",
       "    'ref': 'G',\n",
       "    'segway': 'GE1',\n",
       "    'tf': {'bs': 3, 'bs_peaks': 3, 'bs_peaks_max': 115.3137},\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 3,\n",
       "     'pos': ['227', '275', '167', '193'],\n",
       "     'ref': 'W',\n",
       "     'refcodon': 'TGG'},\n",
       "    'alt': 'A',\n",
       "    'ancestral_allele': 'G',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.50716,\n",
       "     'score': 0.9904489076876952},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 11.76209,\n",
       "     'raw': 0.926699615331582},\n",
       "    'eigen-pc': {'phred': 10.03928,\n",
       "     'raw': 0.797126696617316,\n",
       "     'raw_rankscore': 0.89252},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFDBHCIJ',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.56866,\n",
       "     'coding_score': 0.93022},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.98309, 'score': 1.0},\n",
       "    'gerp++': {'nr': 5.22, 'rs': 5.22, 'rs_rankscore': 0.72176},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8784,\n",
       "     'fitcons_score': 0.724815},\n",
       "    'hg18': {'end': 54651187, 'start': 54651187},\n",
       "    'hg19': {'end': 56364920, 'start': 56364920},\n",
       "    'hg38': {'end': 55971136, 'start': 55971136},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.83238,\n",
       "     'fitcons_score': 0.714379},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.72963,\n",
       "     'fitcons_score': 0.706548},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.62918,\n",
       "     'omega': 0.062186,\n",
       "     'pred': 'D',\n",
       "     'score': 1e-06},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['W193*', 'W167*', 'W275*', 'W227*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 1.0, 'mammalian_rankscore': 0.88762}},\n",
       "    'phylo': {'p100way': {'vertebrate': 8.139,\n",
       "      'vertebrate_rankscore': 0.89444},\n",
       "     'p20way': {'mammalian': 0.978, 'mammalian_rankscore': 0.59618}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'G',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 17.9463,\n",
       "     'logodds_rankscore': 0.88907,\n",
       "     'pi': {'a': 0.0, 'c': 0.0, 'g': 1.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56364920, 'start': 56364920},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '921'},\n",
       "      'cds': {'length': '897', 'position': '681'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.681G>A',\n",
       "      'hgvs_p': 'p.Trp227*',\n",
       "      'protein': {'length': '298', 'position': '227'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '6',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '741'},\n",
       "      'cds': {'length': '717', 'position': '501'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.501G>A',\n",
       "      'hgvs_p': 'p.Trp167*',\n",
       "      'protein': {'length': '238', 'position': '167'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '4',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '819'},\n",
       "      'cds': {'length': '795', 'position': '579'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.579G>A',\n",
       "      'hgvs_p': 'p.Trp193*',\n",
       "      'protein': {'length': '264', 'position': '193'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '5',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2777',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252036.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-181G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4424',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2875',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252037.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2875',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_002868.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'A', 'position': '56364920', 'ref': 'G'}}],\n",
       " 'max_score': 8.73773,\n",
       " 'took': 7,\n",
       " 'total': 5687}"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# query for all variants in MyVariant.info related to gene CDK2 using myvariant python package\n",
    "mv.query('clinvar.gene.symbol:CDK2 OR \\\n",
    "          docm.genename:CDK2 OR \\\n",
    "          snpeff.nmd.genename:CDK2 OR \\\n",
    "          gwassnps.genename:CDK2 OR \\\n",
    "          dbnsfp.genename:CDK2 OR \\\n",
    "          docm.default_gene_name:CDK2 OR \\\n",
    "          snpeff.ann.genename:CDK2 OR \\\n",
    "          snpeff.lof.genename:CDK2 OR \\\n",
    "          emv.gene.symbol:CDK2 OR \\\n",
    "          dbsnp.gene.symbol:CDK2 OR \\\n",
    "          evs.gene.symbol:CDK2 OR \\\n",
    "          cadd.gene.genename:CDK2', fetch_all=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Approach 2: Using JSON-LD powered neutral query function"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "from biothings_helper import query_biothings"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "clinvar.gene.symbol:CDK2 OR docm.genename:CDK2 OR snpeff.nmd.genename:CDK2 OR gwassnps.genename:CDK2 OR dbnsfp.genename:CDK2 OR docm.default_gene_name:CDK2 OR snpeff.ann.genename:CDK2 OR snpeff.lof.genename:CDK2 OR emv.gene.symbol:CDK2 OR dbsnp.gene.symbol:CDK2 OR evs.gene.symbol:CDK2 OR cadd.gene.genename:CDK2\n",
      "None\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'hits': [{'_id': 'chr12:g.56361693G>T',\n",
       "   '_score': 8.73773,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'G',\n",
       "    'annotype': ['CodingTranscript', 'Intergenic', 'Intergenic'],\n",
       "    'bstatistic': 251,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.024,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.039,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.142,\n",
       "     'tssaflnk': 0.291,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.0,\n",
       "     'txflnk': 0.063,\n",
       "     'txwk': 0.181,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': ['stop_gained', 'upstream', 'upstream'],\n",
       "    'consequence': ['STOP_GAINED', 'UPSTREAM', 'UPSTREAM'],\n",
       "    'consscore': [8, 1, 1],\n",
       "    'cpg': 0.03,\n",
       "    'dna': {'helt': 0.43, 'mgw': 0.6, 'prot': -1.24, 'roll': 6.36},\n",
       "    'encode': {'exp': 521.18,\n",
       "     'h3k27ac': 20.96,\n",
       "     'h3k4me1': 29.52,\n",
       "     'h3k4me3': 55.04,\n",
       "     'nucleo': 2.1},\n",
       "    'exon': '2/7',\n",
       "    'fitcons': 0.623825,\n",
       "    'gc': 0.48,\n",
       "    'gene': [{'ccds_id': 'CCDS8898.1',\n",
       "      'cds': {'cdna_pos': 409,\n",
       "       'cds_pos': 169,\n",
       "       'rel_cdna_pos': 0.18,\n",
       "       'rel_cds_pos': 0.19},\n",
       "      'feature_id': 'ENST00000266970',\n",
       "      'gene_id': 'ENSG00000123374',\n",
       "      'genename': 'CDK2',\n",
       "      'prot': {'domain': 'ndomain', 'protpos': 57, 'rel_prot_pos': 0.19}},\n",
       "     {'ccds_id': 'CCDS55834.1',\n",
       "      'feature_id': 'ENST00000449260',\n",
       "      'gene_id': 'ENSG00000185664',\n",
       "      'genename': 'PMEL'},\n",
       "     {'feature_id': 'ENST00000554022',\n",
       "      'gene_id': 'ENSG00000258554',\n",
       "      'genename': 'RP11-973D8.4'}],\n",
       "    'gerp': {'n': 4.73, 'rs': 406.8, 'rs_pval': 1.65278e-39, 's': 4.73},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 403,\n",
       "    'min_dist_tss': 435,\n",
       "    'mutindex': -42,\n",
       "    'naa': '*',\n",
       "    'oaa': 'E',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.996, 'vertebrate': 1.0},\n",
       "    'phred': 38,\n",
       "    'phylop': {'mammalian': 2.626, 'primate': 0.486, 'vertebrate': 5.559},\n",
       "    'pos': 56361693,\n",
       "    'rawscore': 11.874943,\n",
       "    'ref': 'G',\n",
       "    'segway': 'E/GM',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 1,\n",
       "     'pos': 57,\n",
       "     'ref': 'E',\n",
       "     'refcodon': 'GAG'},\n",
       "    'alt': 'T',\n",
       "    'ancestral_allele': 'G',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.7981,\n",
       "     'score': 0.9969183004977041},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 22.49633,\n",
       "     'raw': 1.18564590828305},\n",
       "    'eigen-pc': {'phred': 20.46959,\n",
       "     'raw': 1.02513172599092,\n",
       "     'raw_rankscore': 0.9855},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970', 'ENSP00000452514', 'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000354056']},\n",
       "    'exac': {'ac': 1,\n",
       "     'adj_ac': 1,\n",
       "     'adj_af': 8.238e-06,\n",
       "     'af': 8.236e-06,\n",
       "     'afr_ac': 0,\n",
       "     'afr_af': 0,\n",
       "     'amr_ac': 0,\n",
       "     'amr_af': 0,\n",
       "     'eas_ac': 0,\n",
       "     'eas_af': 0,\n",
       "     'fin_ac': 0,\n",
       "     'fin_af': 0,\n",
       "     'nfe_ac': 1,\n",
       "     'nfe_af': 1.499e-05,\n",
       "     'sas_ac': 0,\n",
       "     'sas_af': 0},\n",
       "    'exac_nontcga': {'ac': 1,\n",
       "     'adj_ac': 1,\n",
       "     'adj_af': 9.417e-06,\n",
       "     'af': 9.415e-06,\n",
       "     'afr_ac': 0,\n",
       "     'afr_af': 0,\n",
       "     'amr_ac': 0,\n",
       "     'amr_af': 0,\n",
       "     'eas_ac': 0,\n",
       "     'eas_af': 0,\n",
       "     'fin_ac': 0,\n",
       "     'fin_af': 0,\n",
       "     'nfe_ac': 1,\n",
       "     'nfe_af': 1.84e-05,\n",
       "     'sas_ac': 0,\n",
       "     'sas_af': 0},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFBCI',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.70836,\n",
       "     'coding_score': 0.96805},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999999999985},\n",
       "    'gerp++': {'nr': 4.73, 'rs': 4.73, 'rs_rankscore': 0.59331},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.61804,\n",
       "     'fitcons_score': 0.685571},\n",
       "    'hg18': {'end': 54647960, 'start': 54647960},\n",
       "    'hg19': {'end': 56361693, 'start': 56361693},\n",
       "    'hg38': {'end': 55967909, 'start': 55967909},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.59914,\n",
       "     'fitcons_score': 0.651492},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.49028,\n",
       "     'fitcons_score': 0.660085},\n",
       "    'interpro_domain': 'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain',\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['E57*', 'E57*', 'E57*', '.'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'without_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'D'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 1.0, 'mammalian_rankscore': 0.88762}},\n",
       "    'phylo': {'p100way': {'vertebrate': 9.985,\n",
       "      'vertebrate_rankscore': 0.99176},\n",
       "     'p20way': {'mammalian': 0.978, 'mammalian_rankscore': 0.59618}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'G',\n",
       "    'rsid': 'rs757201758',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 17.0098,\n",
       "     'logodds_rankscore': 0.86213,\n",
       "     'pi': {'a': 0.0, 'c': 0.0, 'g': 1.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'dbsnp': {'allele_origin': 'unspecified',\n",
       "    'alleles': [{'allele': 'G'}, {'allele': 'T'}],\n",
       "    'alt': 'T',\n",
       "    'chrom': '12',\n",
       "    'class': 'SNV',\n",
       "    'dbsnp_build': 144,\n",
       "    'flags': ['ASP', 'INT', 'NSN', 'R5', 'REF'],\n",
       "    'gene': [{'geneid': '1017', 'symbol': 'CDK2'},\n",
       "     {'geneid': '6490', 'symbol': 'PMEL'}],\n",
       "    'hg19': {'end': 56361693, 'start': 56361693},\n",
       "    'ref': 'G',\n",
       "    'rsid': 'rs757201758',\n",
       "    'validated': False,\n",
       "    'var_subtype': 'tv',\n",
       "    'vartype': 'snp'},\n",
       "   'exac': {'ac': {'ac': 1,\n",
       "     'ac_adj': 1,\n",
       "     'ac_afr': 0,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 0,\n",
       "     'ac_female': 1,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 1,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 0,\n",
       "     'ac_nfe': 1,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 8.236e-06,\n",
       "    'alleles': 'T',\n",
       "    'alt': 'T',\n",
       "    'an': {'an': 121412,\n",
       "     'an_adj': 121396,\n",
       "     'an_afr': 10406,\n",
       "     'an_amr': 11572,\n",
       "     'an_eas': 8654,\n",
       "     'an_female': 54114,\n",
       "     'an_fin': 6614,\n",
       "     'an_male': 67282,\n",
       "     'an_nfe': 66732,\n",
       "     'an_oth': 906,\n",
       "     'an_sas': 16512},\n",
       "    'baseqranksum': -2.257,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.394,\n",
       "    'culprit': 'MQRankSum',\n",
       "    'fs': 2.112,\n",
       "    'het': {'het_afr': 0,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 0,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 1,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': 0.0004,\n",
       "    'mq': {'mq': 59.71, 'mq0': 0, 'mqranksum': -0.856},\n",
       "    'ncc': 7,\n",
       "    'pos': 56361693,\n",
       "    'qd': 11.01,\n",
       "    'readposranksum': 0.802,\n",
       "    'ref': 'G',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 0.69},\n",
       "   'exac_nontcga': {'ac': {'ac': 1,\n",
       "     'ac_adj': 1,\n",
       "     'ac_afr': 0,\n",
       "     'ac_amr': 0,\n",
       "     'ac_eas': 0,\n",
       "     'ac_female': 1,\n",
       "     'ac_fin': 0,\n",
       "     'ac_het': 1,\n",
       "     'ac_hom': 0,\n",
       "     'ac_male': 0,\n",
       "     'ac_nfe': 1,\n",
       "     'ac_oth': 0,\n",
       "     'ac_sas': 0},\n",
       "    'af': 9.415e-06,\n",
       "    'alleles': 'T',\n",
       "    'alt': 'T',\n",
       "    'an': {'an': 106210,\n",
       "     'an_adj': 106196,\n",
       "     'an_afr': 9066,\n",
       "     'an_amr': 11210,\n",
       "     'an_eas': 7866,\n",
       "     'an_female': 45866,\n",
       "     'an_fin': 6614,\n",
       "     'an_male': 60330,\n",
       "     'an_nfe': 54338,\n",
       "     'an_oth': 694,\n",
       "     'an_sas': 16408},\n",
       "    'baseqranksum': -2.257,\n",
       "    'chrom': 12,\n",
       "    'clippingranksum': -0.394,\n",
       "    'culprit': 'MQRankSum',\n",
       "    'fs': 2.112,\n",
       "    'het': {'het_afr': 0,\n",
       "     'het_amr': 0,\n",
       "     'het_eas': 0,\n",
       "     'het_fin': 0,\n",
       "     'het_nfe': 1,\n",
       "     'het_oth': 0,\n",
       "     'het_sas': 0},\n",
       "    'hom': {'hom_afr': 0,\n",
       "     'hom_amr': 0,\n",
       "     'hom_eas': 0,\n",
       "     'hom_fin': 0,\n",
       "     'hom_nfe': 0,\n",
       "     'hom_oth': 0,\n",
       "     'hom_sas': 0},\n",
       "    'inbreedingcoeff': 0.0004,\n",
       "    'mq': {'mq': 59.71, 'mq0': 0, 'mqranksum': -0.856},\n",
       "    'ncc': 7,\n",
       "    'pos': 56361693,\n",
       "    'qd': 11.01,\n",
       "    'readposranksum': 0.802,\n",
       "    'ref': 'G',\n",
       "    'type': 'snp',\n",
       "    'vqslod': 0.69},\n",
       "   'hg19': {'end': 56361693, 'start': 56361693},\n",
       "   'observed': True,\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '409'},\n",
       "      'cds': {'length': '897', 'position': '169'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.169G>T',\n",
       "      'hgvs_p': 'p.Glu57*',\n",
       "      'protein': {'length': '298', 'position': '57'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '2',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '409'},\n",
       "      'cds': {'length': '795', 'position': '169'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.169G>T',\n",
       "      'hgvs_p': 'p.Glu57*',\n",
       "      'protein': {'length': '264', 'position': '57'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '2',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '1847',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52C>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '1847',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52C>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '1197',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87C>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.117-140G>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '1',\n",
       "      'total': '4',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '0.67'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '0.67'}},\n",
       "   'vcf': {'alt': 'T', 'position': '56361693', 'ref': 'G'}},\n",
       "  {'_id': 'chr12:g.56362574C>T',\n",
       "   '_score': 7.2263837,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'C',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 250,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.071,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.016,\n",
       "     'tssaflnk': 0.008,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.15,\n",
       "     'txflnk': 0.039,\n",
       "     'txwk': 0.646,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.03,\n",
       "    'dna': {'helt': 1.21, 'mgw': 0.62, 'prot': -2.97, 'roll': 1.34},\n",
       "    'dst2splice': 13,\n",
       "    'dst2spltype': 'ACCEPTOR',\n",
       "    'encode': {'exp': 193.32,\n",
       "     'h3k27ac': 7.16,\n",
       "     'h3k4me1': 14.12,\n",
       "     'h3k4me3': 19.96,\n",
       "     'nucleo': 2.2},\n",
       "    'exon': '4/7',\n",
       "    'fitcons': 0.726386,\n",
       "    'gc': 0.5,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 568,\n",
       "      'cds_pos': 328,\n",
       "      'rel_cdna_pos': 0.25,\n",
       "      'rel_cds_pos': 0.37},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 110, 'rel_prot_pos': 0.37}},\n",
       "    'gerp': {'n': 4.53, 'rs': 602.1, 'rs_pval': 2.56976e-59, 's': 4.53},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'FALSE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 479,\n",
       "    'min_dist_tss': 1328,\n",
       "    'mutindex': -26,\n",
       "    'naa': '*',\n",
       "    'oaa': 'Q',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.985, 'vertebrate': 1.0},\n",
       "    'phred': 39,\n",
       "    'phylop': {'mammalian': 2.522, 'primate': 0.457, 'vertebrate': 5.459},\n",
       "    'pos': 56362574,\n",
       "    'rawscore': 12.632847,\n",
       "    'ref': 'C',\n",
       "    'segway': 'GE1',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 1,\n",
       "     'pos': ['110', '110', '84', '110'],\n",
       "     'ref': 'Q',\n",
       "     'refcodon': 'CAG'},\n",
       "    'alt': 'T',\n",
       "    'ancestral_allele': 'C',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.89712,\n",
       "     'score': 0.9981459634050125},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 17.64166,\n",
       "     'raw': 1.10312033626684},\n",
       "    'eigen-pc': {'phred': 15.25602,\n",
       "     'raw': 0.933325513156562,\n",
       "     'raw_rankscore': 0.96387},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFBI',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.77402,\n",
       "     'coding_score': 0.97845},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.99999999999996},\n",
       "    'gerp++': {'nr': 4.53, 'rs': 4.53, 'rs_rankscore': 0.54846},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8784,\n",
       "     'fitcons_score': 0.724815},\n",
       "    'hg18': {'end': 54648841, 'start': 54648841},\n",
       "    'hg19': {'end': 56362574, 'start': 56362574},\n",
       "    'hg38': {'end': 55968790, 'start': 55968790},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.9842,\n",
       "     'fitcons_score': 0.735409},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8535,\n",
       "     'fitcons_score': 0.722319},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['Q110*', 'Q84*', 'Q110*', 'Q110*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 1.0, 'mammalian_rankscore': 0.88762}},\n",
       "    'phylo': {'p100way': {'vertebrate': 7.866,\n",
       "      'vertebrate_rankscore': 0.85362},\n",
       "     'p20way': {'mammalian': 0.886, 'mammalian_rankscore': 0.38909}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'C',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 16.5654,\n",
       "     'logodds_rankscore': 0.84367,\n",
       "     'pi': {'a': 0.0, 'c': 1.0, 'g': 0.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56362574, 'start': 56362574},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '568'},\n",
       "      'cds': {'length': '897', 'position': '328'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.328C>T',\n",
       "      'hgvs_p': 'p.Gln110*',\n",
       "      'protein': {'length': '298', 'position': '110'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '4',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '490'},\n",
       "      'cds': {'length': '717', 'position': '250'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.250C>T',\n",
       "      'hgvs_p': 'p.Gln84*',\n",
       "      'protein': {'length': '238', 'position': '84'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '568'},\n",
       "      'cds': {'length': '795', 'position': '328'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.328C>T',\n",
       "      'hgvs_p': 'p.Gln110*',\n",
       "      'protein': {'length': '264', 'position': '110'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '4',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2728',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2728',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2078',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'T', 'position': '56362574', 'ref': 'C'}},\n",
       "  {'_id': 'chr12:g.56361903A>T',\n",
       "   '_score': 7.2185206,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'A',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 251,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.055,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.047,\n",
       "     'tssaflnk': 0.094,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.0,\n",
       "     'txflnk': 0.11,\n",
       "     'txwk': 0.457,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.0,\n",
       "    'dna': {'helt': -3.16, 'mgw': 2.28, 'prot': 4.14, 'roll': 13.24},\n",
       "    'encode': {'exp': 696.29,\n",
       "     'h3k27ac': 10.16,\n",
       "     'h3k4me1': 19.92,\n",
       "     'h3k4me3': 29.88,\n",
       "     'nucleo': 1.1},\n",
       "    'exon': '3/7',\n",
       "    'fitcons': 0.623825,\n",
       "    'gc': 0.44,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 505,\n",
       "      'cds_pos': 265,\n",
       "      'rel_cdna_pos': 0.22,\n",
       "      'rel_cds_pos': 0.3},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 89, 'rel_prot_pos': 0.3}},\n",
       "    'gerp': {'n': 5.26, 'rs': 504.2, 'rs_pval': 6.27736e-64, 's': 5.26},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 193,\n",
       "    'min_dist_tss': 645,\n",
       "    'mutindex': -43,\n",
       "    'naa': '*',\n",
       "    'oaa': 'K',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.995, 'vertebrate': 1.0},\n",
       "    'phred': 40,\n",
       "    'phylop': {'mammalian': 2.122, 'primate': 0.398, 'vertebrate': 3.1},\n",
       "    'pos': 56361903,\n",
       "    'rawscore': 12.734282,\n",
       "    'ref': 'A',\n",
       "    'segway': 'E/GM',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 1,\n",
       "     'pos': ['89', '89', '63', '89'],\n",
       "     'ref': 'K',\n",
       "     'refcodon': 'AAA'},\n",
       "    'alt': 'T',\n",
       "    'ancestral_allele': 'A',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.72936,\n",
       "     'score': 0.9958366694956521},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 9.623577,\n",
       "     'raw': 0.840452296876286},\n",
       "    'eigen-pc': {'phred': 8.147482,\n",
       "     'raw': 0.720175393750942,\n",
       "     'raw_rankscore': 0.83692},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFB',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.75326,\n",
       "     'coding_score': 0.97552},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999974230988},\n",
       "    'gerp++': {'nr': 5.26, 'rs': 5.26, 'rs_rankscore': 0.73376},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.61804,\n",
       "     'fitcons_score': 0.685571},\n",
       "    'hg18': {'end': 54648170, 'start': 54648170},\n",
       "    'hg19': {'end': 56361903, 'start': 56361903},\n",
       "    'hg38': {'end': 55968119, 'start': 55968119},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.59914,\n",
       "     'fitcons_score': 0.651492},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.49028,\n",
       "     'fitcons_score': 0.660085},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.049683,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['K89*', 'K63*', 'K89*', 'K89*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 1.0, 'mammalian_rankscore': 0.88762}},\n",
       "    'phylo': {'p100way': {'vertebrate': 9.248, 'vertebrate_rankscore': 0.9458},\n",
       "     'p20way': {'mammalian': 1.118, 'mammalian_rankscore': 0.88034}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'A',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 14.4595,\n",
       "     'logodds_rankscore': 0.66874,\n",
       "     'pi': {'a': 1.0, 'c': 0.0, 'g': 0.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56361903, 'start': 56361903},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '505'},\n",
       "      'cds': {'length': '897', 'position': '265'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.265A>T',\n",
       "      'hgvs_p': 'p.Lys89*',\n",
       "      'protein': {'length': '298', 'position': '89'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '427'},\n",
       "      'cds': {'length': '717', 'position': '187'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.187A>T',\n",
       "      'hgvs_p': 'p.Lys63*',\n",
       "      'protein': {'length': '238', 'position': '63'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '2',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '505'},\n",
       "      'cds': {'length': '795', 'position': '265'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.265A>T',\n",
       "      'hgvs_p': 'p.Lys89*',\n",
       "      'protein': {'length': '264', 'position': '89'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2057',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2057',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '1407',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'T', 'position': '56361903', 'ref': 'A'}},\n",
       "  {'_id': 'chr12:g.56361869C>A',\n",
       "   '_score': 7.1632676,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'A',\n",
       "    'anc': 'C',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 251,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.055,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.047,\n",
       "     'tssaflnk': 0.094,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.0,\n",
       "     'txflnk': 0.11,\n",
       "     'txwk': 0.457,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.0,\n",
       "    'dna': {'helt': 1.06, 'mgw': -0.35, 'prot': -3.67, 'roll': -2.25},\n",
       "    'encode': {'exp': 683.87,\n",
       "     'h3k27ac': 11.52,\n",
       "     'h3k4me1': 21.44,\n",
       "     'h3k4me3': 26.48,\n",
       "     'nucleo': 1.2},\n",
       "    'exon': '3/7',\n",
       "    'fitcons': 0.623825,\n",
       "    'gc': 0.44,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 471,\n",
       "      'cds_pos': 231,\n",
       "      'rel_cdna_pos': 0.21,\n",
       "      'rel_cds_pos': 0.26},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 77, 'rel_prot_pos': 0.26}},\n",
       "    'gerp': {'n': 5.26, 'rs': 504.2, 'rs_pval': 6.27736e-64, 's': 4.36},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 227,\n",
       "    'min_dist_tss': 611,\n",
       "    'mutindex': -2,\n",
       "    'naa': '*',\n",
       "    'oaa': 'Y',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.997, 'vertebrate': 1.0},\n",
       "    'phred': 36,\n",
       "    'phylop': {'mammalian': 1.329, 'primate': 0.486, 'vertebrate': 3.117},\n",
       "    'pos': 56361869,\n",
       "    'rawscore': 10.033274,\n",
       "    'ref': 'C',\n",
       "    'segway': 'E/GM',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 2,\n",
       "     'codonpos': 3,\n",
       "     'pos': ['77', '77', '51', '77'],\n",
       "     'ref': 'Y',\n",
       "     'refcodon': 'TAC'},\n",
       "    'alt': 'A',\n",
       "    'ancestral_allele': 'C',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.82906,\n",
       "     'score': 0.9973468632930617},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 6.40465,\n",
       "     'raw': 0.644141070663874},\n",
       "    'eigen-pc': {'phred': 5.174308,\n",
       "     'raw': 0.501693885127163,\n",
       "     'raw_rankscore': 0.6821},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFB',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.68415,\n",
       "     'coding_score': 0.96333},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999937496681},\n",
       "    'gerp++': {'nr': 5.26, 'rs': 4.36, 'rs_rankscore': 0.51475},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.61804,\n",
       "     'fitcons_score': 0.685571},\n",
       "    'hg18': {'end': 54648136, 'start': 54648136},\n",
       "    'hg19': {'end': 56361869, 'start': 56361869},\n",
       "    'hg38': {'end': 55968085, 'start': 55968085},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.59914,\n",
       "     'fitcons_score': 0.651492},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.49028,\n",
       "     'fitcons_score': 0.660085},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['Y77*', 'Y51*', 'Y77*', 'Y77*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 0.999, 'mammalian_rankscore': 0.74975}},\n",
       "    'phylo': {'p100way': {'vertebrate': 1.024,\n",
       "      'vertebrate_rankscore': 0.29476},\n",
       "     'p20way': {'mammalian': 0.038, 'mammalian_rankscore': 0.15345}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'C',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 12.2421,\n",
       "     'logodds_rankscore': 0.53681,\n",
       "     'pi': {'a': 0.0, 'c': 0.9144, 'g': 0.0, 't': 0.0856}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56361869, 'start': 56361869},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '471'},\n",
       "      'cds': {'length': '897', 'position': '231'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.231C>A',\n",
       "      'hgvs_p': 'p.Tyr77*',\n",
       "      'protein': {'length': '298', 'position': '77'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '393'},\n",
       "      'cds': {'length': '717', 'position': '153'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.153C>A',\n",
       "      'hgvs_p': 'p.Tyr51*',\n",
       "      'protein': {'length': '238', 'position': '51'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '2',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '471'},\n",
       "      'cds': {'length': '795', 'position': '231'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.231C>A',\n",
       "      'hgvs_p': 'p.Tyr77*',\n",
       "      'protein': {'length': '264', 'position': '77'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2023',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52G>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2023',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52G>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '1373',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87G>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'A', 'position': '56361869', 'ref': 'C'}},\n",
       "  {'_id': 'chr12:g.56362567T>G',\n",
       "   '_score': 7.1632676,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'G',\n",
       "    'anc': 'T',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 250,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.071,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.016,\n",
       "     'tssaflnk': 0.008,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.15,\n",
       "     'txflnk': 0.039,\n",
       "     'txwk': 0.646,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.03,\n",
       "    'dna': {'helt': -1.67, 'mgw': 0.25, 'prot': 2.06, 'roll': 0.17},\n",
       "    'dst2splice': 6,\n",
       "    'dst2spltype': 'ACCEPTOR',\n",
       "    'encode': {'exp': 154.41,\n",
       "     'h3k27ac': 7.16,\n",
       "     'h3k4me1': 14.12,\n",
       "     'h3k4me3': 19.96,\n",
       "     'nucleo': 2.8},\n",
       "    'exon': '4/7',\n",
       "    'fitcons': 0.726386,\n",
       "    'gc': 0.52,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 561,\n",
       "      'cds_pos': 321,\n",
       "      'rel_cdna_pos': 0.24,\n",
       "      'rel_cds_pos': 0.36},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 107, 'rel_prot_pos': 0.36}},\n",
       "    'gerp': {'n': 4.53, 'rs': 602.1, 'rs_pval': 2.56976e-59, 's': 2.07},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 472,\n",
       "    'min_dist_tss': 1321,\n",
       "    'mutindex': -13,\n",
       "    'naa': '*',\n",
       "    'oaa': 'Y',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.961, 'vertebrate': 1.0},\n",
       "    'phred': 36,\n",
       "    'phylop': {'mammalian': 0.808, 'primate': 0.375, 'vertebrate': 0.429},\n",
       "    'pos': 56362567,\n",
       "    'rawscore': 9.700482,\n",
       "    'ref': 'T',\n",
       "    'segway': 'GE1',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 2,\n",
       "     'codonpos': 3,\n",
       "     'pos': ['107', '107', '81', '107'],\n",
       "     'ref': 'Y',\n",
       "     'refcodon': 'TAT'},\n",
       "    'alt': 'G',\n",
       "    'ancestral_allele': 'T',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.77255,\n",
       "     'score': 0.996532737764271},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 5.131892,\n",
       "     'raw': 0.512233895100652},\n",
       "    'eigen-pc': {'phred': 3.893138,\n",
       "     'raw': 0.329494801375175,\n",
       "     'raw_rankscore': 0.57571},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFBI',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.48831,\n",
       "     'coding_score': 0.8894},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.42663, 'score': 0.999765088166355},\n",
       "    'gerp++': {'nr': 4.53, 'rs': 2.07, 'rs_rankscore': 0.25882},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8784,\n",
       "     'fitcons_score': 0.724815},\n",
       "    'hg18': {'end': 54648834, 'start': 54648834},\n",
       "    'hg19': {'end': 56362567, 'start': 56362567},\n",
       "    'hg38': {'end': 55968783, 'start': 55968783},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.9842,\n",
       "     'fitcons_score': 0.735409},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8535,\n",
       "     'fitcons_score': 0.722319},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['Y107*', 'Y81*', 'Y107*', 'Y107*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 1.0, 'mammalian_rankscore': 0.88762}},\n",
       "    'phylo': {'p100way': {'vertebrate': 1.715, 'vertebrate_rankscore': 0.3736},\n",
       "     'p20way': {'mammalian': 1.004, 'mammalian_rankscore': 0.6225}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'T',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 8.4104,\n",
       "     'logodds_rankscore': 0.31622,\n",
       "     'pi': {'a': 0.0, 'c': 0.1901, 'g': 0.0, 't': 0.8099}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56362567, 'start': 56362567},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '561'},\n",
       "      'cds': {'length': '897', 'position': '321'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.321T>G',\n",
       "      'hgvs_p': 'p.Tyr107*',\n",
       "      'protein': {'length': '298', 'position': '107'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '4',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '483'},\n",
       "      'cds': {'length': '717', 'position': '243'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.243T>G',\n",
       "      'hgvs_p': 'p.Tyr81*',\n",
       "      'protein': {'length': '238', 'position': '81'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '561'},\n",
       "      'cds': {'length': '795', 'position': '321'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.321T>G',\n",
       "      'hgvs_p': 'p.Tyr107*',\n",
       "      'protein': {'length': '264', 'position': '107'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '4',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2721',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52A>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2721',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52A>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2071',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87A>C',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'G', 'position': '56362567', 'ref': 'T'}},\n",
       "  {'_id': 'chr12:g.56361900A>T',\n",
       "   '_score': 7.123927,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'A',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 251,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.055,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.047,\n",
       "     'tssaflnk': 0.094,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.0,\n",
       "     'txflnk': 0.11,\n",
       "     'txwk': 0.457,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.0,\n",
       "    'dna': {'helt': -0.9, 'mgw': 0.88, 'prot': 3.28, 'roll': 7.71},\n",
       "    'encode': {'exp': 708.71,\n",
       "     'h3k27ac': 9.44,\n",
       "     'h3k4me1': 22.4,\n",
       "     'h3k4me3': 24.48,\n",
       "     'nucleo': 1.1},\n",
       "    'exon': '3/7',\n",
       "    'fitcons': 0.623825,\n",
       "    'gc': 0.44,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 502,\n",
       "      'cds_pos': 262,\n",
       "      'rel_cdna_pos': 0.22,\n",
       "      'rel_cds_pos': 0.29},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 88, 'rel_prot_pos': 0.3}},\n",
       "    'gerp': {'n': 5.26, 'rs': 504.2, 'rs_pval': 6.27736e-64, 's': 5.26},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 196,\n",
       "    'min_dist_tss': 642,\n",
       "    'mutindex': -36,\n",
       "    'naa': '*',\n",
       "    'oaa': 'K',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.994, 'vertebrate': 1.0},\n",
       "    'phred': 38,\n",
       "    'phylop': {'mammalian': 2.122, 'primate': 0.398, 'vertebrate': 4.77},\n",
       "    'pos': 56361900,\n",
       "    'rawscore': 11.826591,\n",
       "    'ref': 'A',\n",
       "    'segway': 'E/GM',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 1,\n",
       "     'pos': ['88', '88', '62', '88'],\n",
       "     'ref': 'K',\n",
       "     'refcodon': 'AAG'},\n",
       "    'alt': 'T',\n",
       "    'ancestral_allele': 'A',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.76439,\n",
       "     'score': 0.9964034676927931},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 9.620236,\n",
       "     'raw': 0.840301643323044},\n",
       "    'eigen-pc': {'phred': 8.143378,\n",
       "     'raw': 0.719973840873594,\n",
       "     'raw_rankscore': 0.8367},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFB',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.83996,\n",
       "     'coding_score': 0.98561},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999999997431},\n",
       "    'gerp++': {'nr': 5.26, 'rs': 5.26, 'rs_rankscore': 0.73376},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.61804,\n",
       "     'fitcons_score': 0.685571},\n",
       "    'hg18': {'end': 54648167, 'start': 54648167},\n",
       "    'hg19': {'end': 56361900, 'start': 56361900},\n",
       "    'hg38': {'end': 55968116, 'start': 55968116},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.59914,\n",
       "     'fitcons_score': 0.651492},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.49028,\n",
       "     'fitcons_score': 0.660085},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['K88*', 'K62*', 'K88*', 'K88*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 0.998, 'mammalian_rankscore': 0.69722}},\n",
       "    'phylo': {'p100way': {'vertebrate': 9.248, 'vertebrate_rankscore': 0.9458},\n",
       "     'p20way': {'mammalian': 1.118, 'mammalian_rankscore': 0.88034}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'A',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 14.4595,\n",
       "     'logodds_rankscore': 0.66874,\n",
       "     'pi': {'a': 1.0, 'c': 0.0, 'g': 0.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56361900, 'start': 56361900},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '502'},\n",
       "      'cds': {'length': '897', 'position': '262'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.262A>T',\n",
       "      'hgvs_p': 'p.Lys88*',\n",
       "      'protein': {'length': '298', 'position': '88'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '424'},\n",
       "      'cds': {'length': '717', 'position': '184'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.184A>T',\n",
       "      'hgvs_p': 'p.Lys62*',\n",
       "      'protein': {'length': '238', 'position': '62'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '2',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '502'},\n",
       "      'cds': {'length': '795', 'position': '262'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.262A>T',\n",
       "      'hgvs_p': 'p.Lys88*',\n",
       "      'protein': {'length': '264', 'position': '88'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2054',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2054',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '1404',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'T', 'position': '56361900', 'ref': 'A'}},\n",
       "  {'_id': 'chr12:g.56363304A>T',\n",
       "   '_score': 7.10932,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'A',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 250,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.063,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.0,\n",
       "     'tssaflnk': 0.0,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.268,\n",
       "     'txflnk': 0.024,\n",
       "     'txwk': 0.583,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.04,\n",
       "    'dna': {'helt': -1.69, 'mgw': 1.56, 'prot': 1.27, 'roll': 8.9},\n",
       "    'encode': {'exp': 278.18,\n",
       "     'h3k27ac': 7.28,\n",
       "     'h3k4me1': 7.08,\n",
       "     'h3k4me3': 7.88,\n",
       "     'nucleo': 0.6},\n",
       "    'exon': '5/7',\n",
       "    'fitcons': 0.722197,\n",
       "    'gc': 0.58,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 772,\n",
       "      'cds_pos': 532,\n",
       "      'rel_cdna_pos': 0.34,\n",
       "      'rel_cds_pos': 0.59},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 178, 'rel_prot_pos': 0.6}},\n",
       "    'gerp': {'n': 5.44, 'rs': 457.7, 'rs_pval': 1.6043e-69, 's': 5.44},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 364,\n",
       "    'min_dist_tss': 2058,\n",
       "    'mutindex': -44,\n",
       "    'naa': '*',\n",
       "    'oaa': 'K',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.977, 'vertebrate': 1.0},\n",
       "    'phred': 41,\n",
       "    'phylop': {'mammalian': 2.365, 'primate': 0.525, 'vertebrate': 3.25},\n",
       "    'pos': 56363304,\n",
       "    'rawscore': 13.329179,\n",
       "    'ref': 'A',\n",
       "    'segway': 'TF2',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 1,\n",
       "     'pos': 178,\n",
       "     'ref': 'K',\n",
       "     'refcodon': 'AAA'},\n",
       "    'alt': 'T',\n",
       "    'ancestral_allele': 'A',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.74665,\n",
       "     'score': 0.9961165327581305},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 13.05126,\n",
       "     'raw': 0.971550766170071},\n",
       "    'eigen-pc': {'phred': 11.7538,\n",
       "     'raw': 0.849496518123564,\n",
       "     'raw_rankscore': 0.92577},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970', 'ENSP00000452514'],\n",
       "     'transcriptid': ['ENST00000266970', 'ENST00000553376']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFDBI',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.66943,\n",
       "     'coding_score': 0.96018},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999989307069},\n",
       "    'gerp++': {'nr': 5.44, 'rs': 5.44, 'rs_rankscore': 0.7927},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.82299,\n",
       "     'fitcons_score': 0.723133},\n",
       "    'hg18': {'end': 54649571, 'start': 54649571},\n",
       "    'hg19': {'end': 56363304, 'start': 56363304},\n",
       "    'hg38': {'end': 55969520, 'start': 55969520},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.83238,\n",
       "     'fitcons_score': 0.714379},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.83035,\n",
       "     'fitcons_score': 0.719381},\n",
       "    'interpro_domain': 'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain',\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['K178*', 'K178*', '.', '.'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'without_aae', 'without_aae'],\n",
       "     'pred': ['A', 'A', 'D', 'D'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 0.999, 'mammalian_rankscore': 0.74975}},\n",
       "    'phylo': {'p100way': {'vertebrate': 7.335,\n",
       "      'vertebrate_rankscore': 0.78321},\n",
       "     'p20way': {'mammalian': 1.162, 'mammalian_rankscore': 0.90104}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'A',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 15.1728,\n",
       "     'logodds_rankscore': 0.72451,\n",
       "     'pi': {'a': 1.0, 'c': 0.0, 'g': 0.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56363304, 'start': 56363304},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '772'},\n",
       "      'cds': {'length': '897', 'position': '532'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.532A>T',\n",
       "      'hgvs_p': 'p.Lys178*',\n",
       "      'protein': {'length': '298', 'position': '178'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '5',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '3458',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4393',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252036.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-181A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '3458',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2808',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4491',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252037.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4491',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_002868.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.408+572A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '3',\n",
       "      'total': '4',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'effect': 'intron_variant',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.486+572A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'rank': '4',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '0.33'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '0.33'}},\n",
       "   'vcf': {'alt': 'T', 'position': '56363304', 'ref': 'A'}},\n",
       "  {'_id': 'chr12:g.56361861A>T',\n",
       "   '_score': 7.091785,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'A',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 251,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.055,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.047,\n",
       "     'tssaflnk': 0.094,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.0,\n",
       "     'txflnk': 0.11,\n",
       "     'txwk': 0.457,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.0,\n",
       "    'dna': {'helt': -3.16, 'mgw': 2.28, 'prot': 4.14, 'roll': 13.24},\n",
       "    'encode': {'exp': 678.9,\n",
       "     'h3k27ac': 11.52,\n",
       "     'h3k4me1': 21.44,\n",
       "     'h3k4me3': 26.48,\n",
       "     'nucleo': 1.3},\n",
       "    'exon': '3/7',\n",
       "    'fitcons': 0.623825,\n",
       "    'gc': 0.42,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 463,\n",
       "      'cds_pos': 223,\n",
       "      'rel_cdna_pos': 0.2,\n",
       "      'rel_cds_pos': 0.25},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 75, 'rel_prot_pos': 0.25}},\n",
       "    'gerp': {'n': 5.26, 'rs': 504.2, 'rs_pval': 6.27736e-64, 's': 5.26},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 235,\n",
       "    'min_dist_tss': 603,\n",
       "    'mutindex': -21,\n",
       "    'naa': '*',\n",
       "    'oaa': 'K',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.996, 'vertebrate': 1.0},\n",
       "    'phred': 40,\n",
       "    'phylop': {'mammalian': 2.122, 'primate': 0.398, 'vertebrate': 3.1},\n",
       "    'pos': 56361861,\n",
       "    'rawscore': 12.755175,\n",
       "    'ref': 'A',\n",
       "    'segway': 'E/GM',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 1,\n",
       "     'pos': ['75', '75', '49', '75'],\n",
       "     'ref': 'K',\n",
       "     'refcodon': 'AAA'},\n",
       "    'alt': 'T',\n",
       "    'ancestral_allele': 'A',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.7215,\n",
       "     'score': 0.9957055916952746},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 10.4152,\n",
       "     'raw': 0.874627814114319},\n",
       "    'eigen-pc': {'phred': 8.938035,\n",
       "     'raw': 0.755795664242958,\n",
       "     'raw_rankscore': 0.86307},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFB',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.83661,\n",
       "     'coding_score': 0.98531},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999974230988},\n",
       "    'gerp++': {'nr': 5.26, 'rs': 5.26, 'rs_rankscore': 0.73376},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.61804,\n",
       "     'fitcons_score': 0.685571},\n",
       "    'hg18': {'end': 54648128, 'start': 54648128},\n",
       "    'hg19': {'end': 56361861, 'start': 56361861},\n",
       "    'hg38': {'end': 55968077, 'start': 55968077},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.59914,\n",
       "     'fitcons_score': 0.651492},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.49028,\n",
       "     'fitcons_score': 0.660085},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['K75*', 'K49*', 'K75*', 'K75*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 1.0, 'mammalian_rankscore': 0.88762}},\n",
       "    'phylo': {'p100way': {'vertebrate': 7.456,\n",
       "      'vertebrate_rankscore': 0.79641},\n",
       "     'p20way': {'mammalian': 1.118, 'mammalian_rankscore': 0.88034}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'A',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 14.4595,\n",
       "     'logodds_rankscore': 0.66874,\n",
       "     'pi': {'a': 1.0, 'c': 0.0, 'g': 0.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56361861, 'start': 56361861},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '463'},\n",
       "      'cds': {'length': '897', 'position': '223'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.223A>T',\n",
       "      'hgvs_p': 'p.Lys75*',\n",
       "      'protein': {'length': '298', 'position': '75'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '385'},\n",
       "      'cds': {'length': '717', 'position': '145'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.145A>T',\n",
       "      'hgvs_p': 'p.Lys49*',\n",
       "      'protein': {'length': '238', 'position': '49'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '2',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '463'},\n",
       "      'cds': {'length': '795', 'position': '223'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.223A>T',\n",
       "      'hgvs_p': 'p.Lys75*',\n",
       "      'protein': {'length': '264', 'position': '75'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '3',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2015',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200054.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2015',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001200053.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-52T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '1365',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'T', 'position': '56361861', 'ref': 'A'}},\n",
       "  {'_id': 'chr12:g.56364963A>T',\n",
       "   '_score': 7.091785,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'T',\n",
       "    'anc': 'A',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 241,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.094,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.0,\n",
       "     'tssaflnk': 0.0,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.331,\n",
       "     'txflnk': 0.024,\n",
       "     'txwk': 0.496,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.05,\n",
       "    'dna': {'helt': -0.77, 'mgw': 0.97, 'prot': 2.54, 'roll': 8.9},\n",
       "    'encode': {'exp': 822.54,\n",
       "     'h3k27ac': 9.84,\n",
       "     'h3k4me1': 10.12,\n",
       "     'h3k4me3': 6.6,\n",
       "     'nucleo': 0.7},\n",
       "    'exon': '6/7',\n",
       "    'fitcons': 0.701516,\n",
       "    'gc': 0.53,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 964,\n",
       "      'cds_pos': 724,\n",
       "      'rel_cdna_pos': 0.42,\n",
       "      'rel_cds_pos': 0.81},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 242, 'rel_prot_pos': 0.81}},\n",
       "    'gerp': {'n': 5.12, 'rs': 2645.2, 'rs_pval': 9.82059e-310, 's': 5.12},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'TRUE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 80,\n",
       "    'min_dist_tss': 59,\n",
       "    'mutindex': -32,\n",
       "    'naa': '*',\n",
       "    'oaa': 'K',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.868, 'vertebrate': 1.0},\n",
       "    'phred': 38,\n",
       "    'phylop': {'mammalian': 2.066, 'primate': 0.478, 'vertebrate': 4.714},\n",
       "    'pos': 56364963,\n",
       "    'rawscore': 12.185485,\n",
       "    'ref': 'A',\n",
       "    'segway': 'GE1',\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 1,\n",
       "     'pos': ['242', '290', '182', '208'],\n",
       "     'ref': 'K',\n",
       "     'refcodon': 'AAG'},\n",
       "    'alt': 'T',\n",
       "    'ancestral_allele': 'A',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.72395,\n",
       "     'score': 0.9957443054637624},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 8.504234,\n",
       "     'raw': 0.785569528099217},\n",
       "    'eigen-pc': {'phred': 6.924379,\n",
       "     'raw': 0.650882296532907,\n",
       "     'raw_rankscore': 0.78559},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFBCIJ',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.3842,\n",
       "     'coding_score': 0.78455},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.74713, 'score': 0.999999999999909},\n",
       "    'gerp++': {'nr': 5.12, 'rs': 5.12, 'rs_rankscore': 0.69338},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8784,\n",
       "     'fitcons_score': 0.724815},\n",
       "    'hg18': {'end': 54651230, 'start': 54651230},\n",
       "    'hg19': {'end': 56364963, 'start': 56364963},\n",
       "    'hg38': {'end': 55971179, 'start': 55971179},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.83238,\n",
       "     'fitcons_score': 0.714379},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.72963,\n",
       "     'fitcons_score': 0.706548},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.84324,\n",
       "     'omega': 0.0,\n",
       "     'pred': 'D',\n",
       "     'score': 0.0},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['K208*', 'K182*', 'K290*', 'K242*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 1.0, 'mammalian_rankscore': 0.88762}},\n",
       "    'phylo': {'p100way': {'vertebrate': 4.968,\n",
       "      'vertebrate_rankscore': 0.63211},\n",
       "     'p20way': {'mammalian': 1.118, 'mammalian_rankscore': 0.88034}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'A',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 14.2119,\n",
       "     'logodds_rankscore': 0.65164,\n",
       "     'pi': {'a': 1.0, 'c': 0.0, 'g': 0.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56364963, 'start': 56364963},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '964'},\n",
       "      'cds': {'length': '897', 'position': '724'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.724A>T',\n",
       "      'hgvs_p': 'p.Lys242*',\n",
       "      'protein': {'length': '298', 'position': '242'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '6',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '784'},\n",
       "      'cds': {'length': '717', 'position': '544'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.544A>T',\n",
       "      'hgvs_p': 'p.Lys182*',\n",
       "      'protein': {'length': '238', 'position': '182'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '4',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '862'},\n",
       "      'cds': {'length': '795', 'position': '622'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.622A>T',\n",
       "      'hgvs_p': 'p.Lys208*',\n",
       "      'protein': {'length': '264', 'position': '208'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '5',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2734',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252036.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-181A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4467',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87T>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2832',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252037.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2832',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_002868.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222A>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'T', 'position': '56364963', 'ref': 'A'}},\n",
       "  {'_id': 'chr12:g.56364920G>A',\n",
       "   '_score': 7.0576696,\n",
       "   'cadd': {'_license': 'http://goo.gl/bkpNhq',\n",
       "    'alt': 'A',\n",
       "    'anc': 'G',\n",
       "    'annotype': 'CodingTranscript',\n",
       "    'bstatistic': 241,\n",
       "    'chmm': {'bivflnk': 0.0,\n",
       "     'enh': 0.094,\n",
       "     'enhbiv': 0.0,\n",
       "     'het': 0.0,\n",
       "     'quies': 0.055,\n",
       "     'reprpc': 0.0,\n",
       "     'reprpcwk': 0.0,\n",
       "     'tssa': 0.0,\n",
       "     'tssaflnk': 0.0,\n",
       "     'tssbiv': 0.0,\n",
       "     'tx': 0.331,\n",
       "     'txflnk': 0.024,\n",
       "     'txwk': 0.496,\n",
       "     'znfrpts': 0.0},\n",
       "    'chrom': 12,\n",
       "    'consdetail': 'stop_gained',\n",
       "    'consequence': 'STOP_GAINED',\n",
       "    'consscore': 8,\n",
       "    'cpg': 0.04,\n",
       "    'dna': {'helt': -1.87, 'mgw': 0.07, 'prot': 1.32, 'roll': -0.88},\n",
       "    'encode': {'exp': 602.32,\n",
       "     'h3k27ac': 6.56,\n",
       "     'h3k4me1': 11.0,\n",
       "     'h3k4me3': 6.0,\n",
       "     'nucleo': 2.9},\n",
       "    'exon': '6/7',\n",
       "    'fitcons': 0.701516,\n",
       "    'gc': 0.5,\n",
       "    'gene': {'ccds_id': 'CCDS8898.1',\n",
       "     'cds': {'cdna_pos': 921,\n",
       "      'cds_pos': 681,\n",
       "      'rel_cdna_pos': 0.4,\n",
       "      'rel_cds_pos': 0.76},\n",
       "     'feature_id': 'ENST00000266970',\n",
       "     'gene_id': 'ENSG00000123374',\n",
       "     'genename': 'CDK2',\n",
       "     'prot': {'domain': 'ndomain', 'protpos': 227, 'rel_prot_pos': 0.76}},\n",
       "    'gerp': {'n': 5.22, 'rs': 2645.2, 'rs_pval': 9.82059e-310, 's': 5.22},\n",
       "    'isderived': 'TRUE',\n",
       "    'isknownvariant': 'FALSE',\n",
       "    'istv': 'FALSE',\n",
       "    'length': 0,\n",
       "    'mapability': {'20bp': 1, '35bp': 1},\n",
       "    'min_dist_tse': 37,\n",
       "    'min_dist_tss': 16,\n",
       "    'mutindex': -27,\n",
       "    'naa': '*',\n",
       "    'oaa': 'W',\n",
       "    'phast_cons': {'mammalian': 1.0, 'primate': 0.782, 'vertebrate': 1.0},\n",
       "    'phred': 37,\n",
       "    'phylop': {'mammalian': 2.613, 'primate': 0.587, 'vertebrate': 5.764},\n",
       "    'pos': 56364920,\n",
       "    'rawscore': 11.158933,\n",
       "    'ref': 'G',\n",
       "    'segway': 'GE1',\n",
       "    'tf': {'bs': 3, 'bs_peaks': 3, 'bs_peaks_max': 115.3137},\n",
       "    'type': 'SNV'},\n",
       "   'chrom': '12',\n",
       "   'dbnsfp': {'aa': {'alt': 'X',\n",
       "     'codon_degeneracy': 0,\n",
       "     'codonpos': 3,\n",
       "     'pos': ['227', '275', '167', '193'],\n",
       "     'ref': 'W',\n",
       "     'refcodon': 'TGG'},\n",
       "    'alt': 'A',\n",
       "    'ancestral_allele': 'G',\n",
       "    'cds_strand': '+',\n",
       "    'chrom': '12',\n",
       "    'dann': {'_license': 'https://goo.gl/IeLhCq',\n",
       "     'rankscore': 0.50716,\n",
       "     'score': 0.9904489076876952},\n",
       "    'eigen': {'coding_or_noncoding': 'c',\n",
       "     'phred': 11.76209,\n",
       "     'raw': 0.926699615331582},\n",
       "    'eigen-pc': {'phred': 10.03928,\n",
       "     'raw': 0.797126696617316,\n",
       "     'raw_rankscore': 0.89252},\n",
       "    'ensembl': {'geneid': 'ENSG00000123374',\n",
       "     'proteinid': ['ENSP00000266970',\n",
       "      'ENSP00000452514',\n",
       "      'ENSP00000393605',\n",
       "      'ENSP00000243067'],\n",
       "     'transcriptid': ['ENST00000266970',\n",
       "      'ENST00000553376',\n",
       "      'ENST00000440311',\n",
       "      'ENST00000354056']},\n",
       "    'fathmm': {'score': None},\n",
       "    'fathmm-mkl': {'coding_group': 'AEFDBHCIJ',\n",
       "     'coding_pred': 'D',\n",
       "     'coding_rankscore': 0.56866,\n",
       "     'coding_score': 0.93022},\n",
       "    'genename': 'CDK2',\n",
       "    'genocanyon': {'rankscore': 0.98309, 'score': 1.0},\n",
       "    'gerp++': {'nr': 5.22, 'rs': 5.22, 'rs_rankscore': 0.72176},\n",
       "    'gm12878': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.65041,\n",
       "     'fitcons_score': 0.672317},\n",
       "    'h1-hesc': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.8784,\n",
       "     'fitcons_score': 0.724815},\n",
       "    'hg18': {'end': 54651187, 'start': 54651187},\n",
       "    'hg19': {'end': 56364920, 'start': 56364920},\n",
       "    'hg38': {'end': 55971136, 'start': 55971136},\n",
       "    'huvec': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.83238,\n",
       "     'fitcons_score': 0.714379},\n",
       "    'integrated': {'confidence_value': 0,\n",
       "     'fitcons_rankscore': 0.72963,\n",
       "     'fitcons_score': 0.706548},\n",
       "    'interpro_domain': ['Protein kinase domain|Protein kinase-like domain',\n",
       "     'Protein kinase domain|Protein kinase-like domain|Serine/threonine/dual specificity protein kinase, catalytic  domain'],\n",
       "    'lrt': {'converted_rankscore': 0.62918,\n",
       "     'omega': 0.062186,\n",
       "     'pred': 'D',\n",
       "     'score': 1e-06},\n",
       "    'm_cap_score': {'score': None},\n",
       "    'metalr': {'score': None},\n",
       "    'metasvm': {'score': None},\n",
       "    'mutationassessor': {'score': None},\n",
       "    'mutationtaster': {'AAE': ['W193*', 'W167*', 'W275*', 'W227*'],\n",
       "     'converted_rankscore': 0.81033,\n",
       "     'model': ['complex_aae', 'complex_aae', 'complex_aae', 'complex_aae'],\n",
       "     'pred': ['A', 'A', 'A', 'A'],\n",
       "     'score': [1, 1, 1, 1]},\n",
       "    'phastcons': {'100way': {'vertebrate': 1.0,\n",
       "      'vertebrate_rankscore': 0.71511},\n",
       "     '20way': {'mammalian': 1.0, 'mammalian_rankscore': 0.88762}},\n",
       "    'phylo': {'p100way': {'vertebrate': 8.139,\n",
       "      'vertebrate_rankscore': 0.89444},\n",
       "     'p20way': {'mammalian': 0.978, 'mammalian_rankscore': 0.59618}},\n",
       "    'polyphen2': {'_license': 'http://goo.gl/6Cz4Ae',\n",
       "     'hdiv': {'score': None},\n",
       "     'hvar': {'score': None}},\n",
       "    'provean': {'score': None},\n",
       "    'ref': 'G',\n",
       "    'sift': {'score': None},\n",
       "    'siphy_29way': {'logodds': 17.9463,\n",
       "     'logodds_rankscore': 0.88907,\n",
       "     'pi': {'a': 0.0, 'c': 0.0, 'g': 1.0, 't': 0.0}},\n",
       "    'vest3': {'_license': 'http://goo.gl/jTko4F', 'score': None}},\n",
       "   'hg19': {'end': 56364920, 'start': 56364920},\n",
       "   'snpeff': {'ann': [{'cdna': {'length': '2301', 'position': '921'},\n",
       "      'cds': {'length': '897', 'position': '681'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001798.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.681G>A',\n",
       "      'hgvs_p': 'p.Trp227*',\n",
       "      'protein': {'length': '298', 'position': '227'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '6',\n",
       "      'total': '7',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2121', 'position': '741'},\n",
       "      'cds': {'length': '717', 'position': '501'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_001290230.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.501G>A',\n",
       "      'hgvs_p': 'p.Trp167*',\n",
       "      'protein': {'length': '238', 'position': '167'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '4',\n",
       "      'total': '5',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'cdna': {'length': '2199', 'position': '819'},\n",
       "      'cds': {'length': '795', 'position': '579'},\n",
       "      'effect': 'stop_gained',\n",
       "      'feature_id': 'NM_052827.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'CDK2',\n",
       "      'genename': 'CDK2',\n",
       "      'hgvs_c': 'c.579G>A',\n",
       "      'hgvs_p': 'p.Trp193*',\n",
       "      'protein': {'length': '264', 'position': '193'},\n",
       "      'putative_impact': 'HIGH',\n",
       "      'rank': '5',\n",
       "      'total': '6',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2777',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252036.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-181G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '4424',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_006928.4',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'PMEL',\n",
       "      'genename': 'PMEL',\n",
       "      'hgvs_c': 'c.-87C>T',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2875',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_001252037.1',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'},\n",
       "     {'distance_to_feature': '2875',\n",
       "      'effect': 'upstream_gene_variant',\n",
       "      'feature_id': 'NM_002868.3',\n",
       "      'feature_type': 'transcript',\n",
       "      'gene_id': 'RAB5B',\n",
       "      'genename': 'RAB5B',\n",
       "      'hgvs_c': 'c.-222G>A',\n",
       "      'putative_impact': 'MODIFIER',\n",
       "      'transcript_biotype': 'Coding'}],\n",
       "    'lof': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'},\n",
       "    'nmd': {'gene_id': 'CDK2',\n",
       "     'genename': 'CDK2',\n",
       "     'number_of_transcripts_in_gene': '3',\n",
       "     'percent_of_transcripts_affected': '1.00'}},\n",
       "   'vcf': {'alt': 'A', 'position': '56364920', 'ref': 'G'}}],\n",
       " 'max_score': 8.73773,\n",
       " 'took': 34,\n",
       " 'total': 5687}"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Query for a gene symbol CDK2 using URI for gene symbol, which is 'http://identifiers.org/hgnc.symbol/'\n",
    "query_biothings(api='myvariant.info', fields_uri='http://identifiers.org/hgnc.symbol/', fields_value='CDK2', fetch_all=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# 2. Facilitate API Cross Linking\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "##### Use Case Scenario: Upstream analysis identified a missense variant (chr8:g.99440236C>A). The analyst wants to obtain the matching InterPro data in order to assess the likely functional significance of this variant."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Step 1: Query MyVariant.info to retrieve the annotation object for variant chr8:g.99440236C>A\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "from biothings_helper import Biothingsexplorer\n",
    "explorer = Biothingsexplorer()\n",
    "json_doc = explorer.get_json_doc('myvariant.info', 'chr8:g.99440236C>A')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'@context': {'cadd.gene.ccds_id': 'http://identifiers.org/ccds/',\n",
       "  'cadd.gene.feature_id': 'http://identifiers.org/ensembl.transcript/',\n",
       "  'cadd.gene.gene_id': 'http://identifiers.org/ensembl.gene/',\n",
       "  'cadd.gene.genename': 'http://identifiers.org/hgnc.symbol/',\n",
       "  'clinvar.gene.id': 'http://identifiers.org/hgnc/',\n",
       "  'clinvar.gene.symbol': 'http://identifiers.org/hgnc.symbol/',\n",
       "  'clinvar.omim': 'http://identifiers.org/omim/',\n",
       "  'clinvar.rcv.accession': 'http://identifers.org/clinvar/',\n",
       "  'clinvar.rcv.conditions.identifiers.efo': 'http://identifiers.org/efo/',\n",
       "  'clinvar.rcv.conditions.identifiers.omim': 'http://identifiers.org/omim/',\n",
       "  'clinvar.rcv.conditions.identifiers.orphanet': 'http://identifiers.org/orphanet/',\n",
       "  'clinvar.rsid': 'http://identifiers.org/dbsnp/',\n",
       "  'clinvar.uniprot': 'http://identifiers.org/uniprot/',\n",
       "  'dbnsfp.clinvar.rs': 'http://identifiers.org/dbsnp/',\n",
       "  'dbnsfp.ensembl.geneid': 'http://identifiers.org/ensembl.gene/',\n",
       "  'dbnsfp.ensembl.proteinid': 'http://identifiers.org/ensembl.protein/',\n",
       "  'dbnsfp.ensembl.transcriptid': 'http://identifiers.org/ensembl.transcript',\n",
       "  'dbnsfp.genename': 'http://identifiers.org/hgnc.symbol/',\n",
       "  'dbnsfp.rsid': 'http://identifiers.org/dbsnp/',\n",
       "  'dbnsfp.uniprot.acc': 'http://identifiers.org/uniprot/',\n",
       "  'dbsnp.gene.geneid': 'http://identifiers.org/hgnc/',\n",
       "  'dbsnp.gene.symbol': 'http://identifiers.org/hgnc.symbol/',\n",
       "  'dbsnp.rsid': 'http://identifiers.org/dbsnp/',\n",
       "  'docm.default_gene_name': 'http://identifiers.org/hgnc.symbol/',\n",
       "  'docm.ensembl_gene_id': 'http://identifiers.org/ensembl.gene/',\n",
       "  'docm.genename': 'http://identifiers.org/hgnc.symbol/',\n",
       "  'docm.pubmed_id': 'http://identifiers.org/pubmed/',\n",
       "  'emv.gene.id': 'http://identifiers.org/hgnc/',\n",
       "  'emv.gene.symbol': 'http://identifiers.org/hgnc.symbol/',\n",
       "  'evs.gene.accession': 'http://identifiers.org/refseq/',\n",
       "  'evs.gene.symbol': 'http://identifiers.org/hgnc.symbol/',\n",
       "  'evs.rsid': 'http://identifiers.org/dbsnp/',\n",
       "  'grasp.publication.snpid': 'http://identifiers.org/dbsnp/',\n",
       "  'grasp.rsid': 'http://identifiers.org/dbsnp/',\n",
       "  'gwassnps.genename': 'http://identifiers.org/hgnc.symbol/',\n",
       "  'gwassnps.pubmed': 'http://identifiers.org/pubmed/',\n",
       "  'gwassnps.rsid': 'http://identifiers.org/dbsnp/',\n",
       "  'mutdb.rsid': 'http://identifiers.org/dbsnp/',\n",
       "  'snpeff.ann.genename': 'http://identifiers.org/hgnc.symbol/',\n",
       "  'snpeff.lof.genename': 'http://identifiers.org/hgnc.symbol/',\n",
       "  'snpeff.nmd.genename': 'http://identifiers.org/hgnc.symbol/'},\n",
       " '_id': 'chr8:g.99440236C>A',\n",
       " '_version': 1,\n",
       " 'cadd._license': 'http://goo.gl/bkpNhq',\n",
       " 'cadd.alt': 'A',\n",
       " 'cadd.anc': 'C',\n",
       " 'cadd.annotype': 'CodingTranscript',\n",
       " 'cadd.bstatistic': 600,\n",
       " 'cadd.chmm.bivflnk': 0.063,\n",
       " 'cadd.chmm.enh': 0.0,\n",
       " 'cadd.chmm.enhbiv': 0.087,\n",
       " 'cadd.chmm.het': 0.016,\n",
       " 'cadd.chmm.quies': 0.039,\n",
       " 'cadd.chmm.reprpc': 0.528,\n",
       " 'cadd.chmm.reprpcwk': 0.094,\n",
       " 'cadd.chmm.tssa': 0.008,\n",
       " 'cadd.chmm.tssaflnk': 0.0,\n",
       " 'cadd.chmm.tssbiv': 0.11,\n",
       " 'cadd.chmm.tx': 0.0,\n",
       " 'cadd.chmm.txflnk': 0.0,\n",
       " 'cadd.chmm.txwk': 0.008,\n",
       " 'cadd.chmm.znfrpts': 0.008,\n",
       " 'cadd.chrom': 8,\n",
       " 'cadd.consdetail': 'stop_gained',\n",
       " 'cadd.consequence': 'STOP_GAINED',\n",
       " 'cadd.consscore': 8,\n",
       " 'cadd.cpg': 0.26,\n",
       " 'cadd.dna.helt': 0.71,\n",
       " 'cadd.dna.mgw': -0.26,\n",
       " 'cadd.dna.prot': -3.57,\n",
       " 'cadd.dna.roll': -1.14,\n",
       " 'cadd.encode.exp': 3.11,\n",
       " 'cadd.encode.h3k27ac': 1.0,\n",
       " 'cadd.encode.h3k4me1': 6.64,\n",
       " 'cadd.encode.h3k4me3': 6.0,\n",
       " 'cadd.encode.nucleo': 0.0,\n",
       " 'cadd.encode.occ': 4,\n",
       " 'cadd.encode.p_val.comb': 15.18,\n",
       " 'cadd.encode.p_val.ctcf': 11.8,\n",
       " 'cadd.encode.p_val.dnas': 15.4,\n",
       " 'cadd.encode.p_val.faire': 1.68,\n",
       " 'cadd.encode.p_val.mycp': 1.62,\n",
       " 'cadd.encode.p_val.polii': 0.0,\n",
       " 'cadd.encode.sig.ctcf': 1.96,\n",
       " 'cadd.encode.sig.dnase': 0.49,\n",
       " 'cadd.encode.sig.faire': 0.02,\n",
       " 'cadd.encode.sig.myc': 0.09,\n",
       " 'cadd.encode.sig.polii': 0.02,\n",
       " 'cadd.exon': '2/2',\n",
       " 'cadd.fitcons': 0.601731,\n",
       " 'cadd.gc': 0.71,\n",
       " 'cadd.gene.ccds_id': 'CCDS6279.1',\n",
       " 'cadd.gene.cds.cdna_pos': 379,\n",
       " 'cadd.gene.cds.cds_pos': 29,\n",
       " 'cadd.gene.cds.rel_cdna_pos': 0.07,\n",
       " 'cadd.gene.cds.rel_cds_pos': 0.02,\n",
       " 'cadd.gene.feature_id': 'ENST00000287042',\n",
       " 'cadd.gene.gene_id': 'ENSG00000156486',\n",
       " 'cadd.gene.genename': 'KCNS2',\n",
       " 'cadd.gene.prot.protpos': 10,\n",
       " 'cadd.gene.prot.rel_prot_pos': 0.02,\n",
       " 'cadd.gerp.n': 4.8,\n",
       " 'cadd.gerp.rs': 6739.9,\n",
       " 'cadd.gerp.rs_pval': 0,\n",
       " 'cadd.gerp.s': 3.91,\n",
       " 'cadd.isderived': 'TRUE',\n",
       " 'cadd.isknownvariant': 'FALSE',\n",
       " 'cadd.istv': 'TRUE',\n",
       " 'cadd.length': 0,\n",
       " 'cadd.mapability.20bp': 1,\n",
       " 'cadd.mapability.35bp': 1,\n",
       " 'cadd.min_dist_tse': 4141,\n",
       " 'cadd.min_dist_tss': 477,\n",
       " 'cadd.mutindex': 47,\n",
       " 'cadd.naa': '*',\n",
       " 'cadd.oaa': 'S',\n",
       " 'cadd.phast_cons.mammalian': 0.633,\n",
       " 'cadd.phast_cons.primate': 0.938,\n",
       " 'cadd.phast_cons.vertebrate': 0.913,\n",
       " 'cadd.phred': 35,\n",
       " 'cadd.phylop.mammalian': 1.222,\n",
       " 'cadd.phylop.primate': 0.552,\n",
       " 'cadd.phylop.vertebrate': 3.762,\n",
       " 'cadd.pos': 99440236,\n",
       " 'cadd.rawscore': 8.943177,\n",
       " 'cadd.ref': 'C',\n",
       " 'cadd.segway': 'TF0',\n",
       " 'cadd.tf.bs': 6,\n",
       " 'cadd.tf.bs_peaks': 7,\n",
       " 'cadd.tf.bs_peaks_max': 142.3577,\n",
       " 'cadd.type': 'SNV',\n",
       " 'chrom': '8',\n",
       " 'dbnsfp.aa.alt': 'X',\n",
       " 'dbnsfp.aa.codon_degeneracy': 0,\n",
       " 'dbnsfp.aa.codonpos': 2,\n",
       " 'dbnsfp.aa.pos': 10,\n",
       " 'dbnsfp.aa.ref': 'S',\n",
       " 'dbnsfp.aa.refcodon': 'TCG',\n",
       " 'dbnsfp.alt': 'A',\n",
       " 'dbnsfp.ancestral_allele': 'c',\n",
       " 'dbnsfp.cds_strand': '+',\n",
       " 'dbnsfp.chrom': '8',\n",
       " 'dbnsfp.dann._license': 'https://goo.gl/IeLhCq',\n",
       " 'dbnsfp.dann.rankscore': 0.56469,\n",
       " 'dbnsfp.dann.score': 0.9924952665873633,\n",
       " 'dbnsfp.eigen-pc.phred': 3.163276,\n",
       " 'dbnsfp.eigen-pc.raw': 0.195921368003061,\n",
       " 'dbnsfp.eigen-pc.raw_rankscore': 0.50005,\n",
       " 'dbnsfp.eigen.coding_or_noncoding': 'c',\n",
       " 'dbnsfp.eigen.phred': 4.558081,\n",
       " 'dbnsfp.eigen.raw': 0.433511512779213,\n",
       " 'dbnsfp.ensembl.geneid': 'ENSG00000156486',\n",
       " 'dbnsfp.ensembl.proteinid': ['ENSP00000287042', 'ENSP00000430712'],\n",
       " 'dbnsfp.ensembl.transcriptid': ['ENST00000287042', 'ENST00000521839'],\n",
       " 'dbnsfp.fathmm-mkl.coding_group': 'AEFDGBHCI',\n",
       " 'dbnsfp.fathmm-mkl.coding_pred': 'N',\n",
       " 'dbnsfp.fathmm-mkl.coding_rankscore': 0.24153,\n",
       " 'dbnsfp.fathmm-mkl.coding_score': 0.316,\n",
       " 'dbnsfp.fathmm.score': None,\n",
       " 'dbnsfp.genename': 'KCNS2',\n",
       " 'dbnsfp.genocanyon.rankscore': 0.74713,\n",
       " 'dbnsfp.genocanyon.score': 0.999999999971059,\n",
       " 'dbnsfp.gerp++.nr': 4.8,\n",
       " 'dbnsfp.gerp++.rs': 3.91,\n",
       " 'dbnsfp.gerp++.rs_rankscore': 0.44212,\n",
       " 'dbnsfp.gm12878.confidence_value': 0,\n",
       " 'dbnsfp.gm12878.fitcons_rankscore': 0.48707,\n",
       " 'dbnsfp.gm12878.fitcons_score': 0.596491,\n",
       " 'dbnsfp.h1-hesc.confidence_value': 2,\n",
       " 'dbnsfp.h1-hesc.fitcons_rankscore': 0.05288,\n",
       " 'dbnsfp.h1-hesc.fitcons_score': 0.378051,\n",
       " 'dbnsfp.hg18.end': 99509412,\n",
       " 'dbnsfp.hg18.start': 99509412,\n",
       " 'dbnsfp.hg19.end': 99440236,\n",
       " 'dbnsfp.hg19.start': 99440236,\n",
       " 'dbnsfp.hg38.end': 98428008,\n",
       " 'dbnsfp.hg38.start': 98428008,\n",
       " 'dbnsfp.huvec.confidence_value': 0,\n",
       " 'dbnsfp.huvec.fitcons_rankscore': 0.58052,\n",
       " 'dbnsfp.huvec.fitcons_score': 0.63947,\n",
       " 'dbnsfp.integrated.confidence_value': 0,\n",
       " 'dbnsfp.integrated.fitcons_rankscore': 0.33991,\n",
       " 'dbnsfp.integrated.fitcons_score': 0.59774,\n",
       " 'dbnsfp.lrt.converted_rankscore': 0.06324,\n",
       " 'dbnsfp.lrt.omega': 1.17275,\n",
       " 'dbnsfp.lrt.pred': 'N',\n",
       " 'dbnsfp.lrt.score': 0.723711,\n",
       " 'dbnsfp.m_cap_score.score': None,\n",
       " 'dbnsfp.metalr.score': None,\n",
       " 'dbnsfp.metasvm.score': None,\n",
       " 'dbnsfp.mutationassessor.score': None,\n",
       " 'dbnsfp.mutationtaster.AAE': ['S10*', 'S10*'],\n",
       " 'dbnsfp.mutationtaster.converted_rankscore': 0.81033,\n",
       " 'dbnsfp.mutationtaster.model': ['complex_aae', 'complex_aae'],\n",
       " 'dbnsfp.mutationtaster.pred': ['D', 'D'],\n",
       " 'dbnsfp.mutationtaster.score': [1, 1],\n",
       " 'dbnsfp.phastcons.100way.vertebrate': 0.04,\n",
       " 'dbnsfp.phastcons.100way.vertebrate_rankscore': 0.20781,\n",
       " 'dbnsfp.phastcons.20way.mammalian': 0.897,\n",
       " 'dbnsfp.phastcons.20way.mammalian_rankscore': 0.37868,\n",
       " 'dbnsfp.phylo.p100way.vertebrate': 2.054,\n",
       " 'dbnsfp.phylo.p100way.vertebrate_rankscore': 0.40723,\n",
       " 'dbnsfp.phylo.p20way.mammalian': 0.033,\n",
       " 'dbnsfp.phylo.p20way.mammalian_rankscore': 0.15068,\n",
       " 'dbnsfp.polyphen2._license': 'http://goo.gl/6Cz4Ae',\n",
       " 'dbnsfp.polyphen2.hdiv.score': None,\n",
       " 'dbnsfp.polyphen2.hvar.score': None,\n",
       " 'dbnsfp.provean.score': None,\n",
       " 'dbnsfp.ref': 'C',\n",
       " 'dbnsfp.rsid': 'rs777451821',\n",
       " 'dbnsfp.sift.score': None,\n",
       " 'dbnsfp.siphy_29way.logodds': 14.2027,\n",
       " 'dbnsfp.siphy_29way.logodds_rankscore': 0.65105,\n",
       " 'dbnsfp.siphy_29way.pi.a': 0.1507,\n",
       " 'dbnsfp.siphy_29way.pi.c': 0.8493,\n",
       " 'dbnsfp.siphy_29way.pi.g': 0.0,\n",
       " 'dbnsfp.siphy_29way.pi.t': 0.0,\n",
       " 'dbnsfp.vest3._license': 'http://goo.gl/jTko4F',\n",
       " 'dbnsfp.vest3.score': None,\n",
       " 'dbsnp.allele_origin': 'unspecified',\n",
       " 'dbsnp.alleles.allele': ['C', 'A', 'G'],\n",
       " 'dbsnp.alt': 'A',\n",
       " 'dbsnp.chrom': '8',\n",
       " 'dbsnp.class': 'SNV',\n",
       " 'dbsnp.dbsnp_build': 144,\n",
       " 'dbsnp.flags': ['ASP', 'NSM', 'NSN', 'REF'],\n",
       " 'dbsnp.gene.geneid': '3788',\n",
       " 'dbsnp.gene.symbol': 'KCNS2',\n",
       " 'dbsnp.hg19.end': 99440236,\n",
       " 'dbsnp.hg19.start': 99440236,\n",
       " 'dbsnp.ref': 'C',\n",
       " 'dbsnp.rsid': 'rs777451821',\n",
       " 'dbsnp.validated': False,\n",
       " 'dbsnp.var_subtype': 'unknown',\n",
       " 'dbsnp.vartype': 'snp',\n",
       " 'hg19.end': 99440236,\n",
       " 'hg19.start': 99440236,\n",
       " 'observed': True,\n",
       " 'snpeff.ann.cdna.length': '3166',\n",
       " 'snpeff.ann.cdna.position': '379',\n",
       " 'snpeff.ann.cds.length': '1434',\n",
       " 'snpeff.ann.cds.position': '29',\n",
       " 'snpeff.ann.effect': 'stop_gained',\n",
       " 'snpeff.ann.feature_id': 'NM_020697.2',\n",
       " 'snpeff.ann.feature_type': 'transcript',\n",
       " 'snpeff.ann.gene_id': 'KCNS2',\n",
       " 'snpeff.ann.genename': 'KCNS2',\n",
       " 'snpeff.ann.hgvs_c': 'c.29C>A',\n",
       " 'snpeff.ann.hgvs_p': 'p.Ser10*',\n",
       " 'snpeff.ann.protein.length': '477',\n",
       " 'snpeff.ann.protein.position': '10',\n",
       " 'snpeff.ann.putative_impact': 'HIGH',\n",
       " 'snpeff.ann.rank': '2',\n",
       " 'snpeff.ann.total': '2',\n",
       " 'snpeff.ann.transcript_biotype': 'Coding',\n",
       " 'vcf.alt': 'A',\n",
       " 'vcf.position': '99440236',\n",
       " 'vcf.ref': 'C'}"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# json doc with json-ld context added\n",
    "json_doc"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Step 2: List all available apis linked from this variant"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Available APIs which could be linked out is: {'mygene.info': 'ENSG00000156486'}\n"
     ]
    }
   ],
   "source": [
    "explorer.find_linked_apis()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Step 3: Link to mygene.info and get interpro ID"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'_id': '3788',\n",
       " '_score': 22.792553,\n",
       " 'interpro': [{'id': 'IPR011333'},\n",
       "  {'id': 'IPR000210'},\n",
       "  {'id': 'IPR005821'},\n",
       "  {'id': 'IPR027359'},\n",
       "  {'id': 'IPR003968'},\n",
       "  {'id': 'IPR028325'},\n",
       "  {'id': 'IPR003131'},\n",
       "  {'id': 'IPR003971'}]}"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "explorer.explore_api('mygene.info', fields_uri='http://identifiers.org/interpro/')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
